<!DOCTYPE html >
<html id="aU9OeVhQDeEGqgTh_58mKV1_4aI8-PMC2483532" data-origid="PMC2483532" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/"/>
    <title>aU9OeVhQDeEGqgTh_58mKV1_4aI8-PMC2483532</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">Building a spindle of the correct length in human cells requires the interaction between TPX2 and Aurora A</h2>
      </section>
      <section data-type="abstract">
        <h2 id="s2h1">Abstract</h2>
        <div class="content">
          <p id="s2p1">To assemble mitotic spindles, cells nucleate microtubules from a variety of sources including chromosomes and centrosomes. We know little about how the regulation of microtubule nucleation contributes to spindle bipolarity and spindle size. The Aurora A kinase activator TPX2 is required for microtubule nucleation from chromosomes as well as for spindle bipolarity. We use bacterial artificial chromosome–based recombineering to introduce point mutants that block the interaction between TPX2 and Aurora A into human cells. TPX2 mutants have very short spindles but, surprisingly, are still bipolar and segregate chromosomes. Examination of microtubule nucleation during spindle assembly shows that microtubules fail to nucleate from chromosomes. Thus, chromosome nucleation is not essential for bipolarity during human cell mitosis when centrosomes are present. Rather, chromosome nucleation is involved in spindle pole separation and setting spindle length. A second Aurora A–independent function of TPX2 is required to bipolarize spindles.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s3h1">Introduction</h2>
        <div class="content">
          <p id="s3p1">When cells assemble a spindle, they nucleate microtubules from various sources, including centrosomes, chromosomes (Heald et al., 1996; Maiato et al., 2004), and the spindle microtubules (Mahoney et al., 2006; for reviews see Luders and Stearns, 2007; O'Connell and Khodjakov, 2007). As these nucleated microtubules are organized into a bipolar spindle, the spindle poles separate a characteristic distance from each other and the chromosomes are aligned midway between them. Thus, we can distinguish two different aspects of forming a spindle: the production of microtubules and their organization into a bipolar array. Although spindles in any particular cell type have very reproducible geometry, we know little about the factors that determine spindle size. One of the interesting questions is the relative role of microtubules nucleated from chromosomes or centrosomes. In some systems, such as Xenopus laevis extracts and Drosophila melanogaster, centrosomes seem dispensable for spindle organization (Heald et al., 1996; Basto et al., 2006), whereas in other systems centrosomes are essential (Hamill et al., 2002). Although chromatin-based microtubule nucleation has been visualized in mammalian cells (Khodjakov et al., 2003), it has been hard to study the role of chromatin/kinetochore-dependent microtubule nucleation in mammalian spindle assembly because specific ways of inhibiting chromatin/kinetochore nucleation have not been identified. Ran, a factor thought responsible for mediating chromosome-based nucleation, is not suitable for study by protein depletion because it is responsible for many cellular processes, including several process within mitosis (for review see Ciciarello et al., 2007). What is needed is a precise way to block growth from chromosomes and to then study the assembly of a spindle.</p>
          <p id="s3p2">One possible component that regulates microtubule growth around chromosomes is the kinase Aurora A. Aurora A has been shown to stimulate nucleation of microtubules in X. laevis extracts when added on beads (Tsai and Zheng, 2005). However, Aurora A has numerous functions in cells (for review see Barr and Gergely, 2007), and it is not established whether Aurora A is indeed required for chromosome nucleation by reduction of function experiments. The specific functions of Aurora A, as with other kinases, are thought to be regulated spatially and temporally by kinase-specific activators. Indeed, several different activators are known for Aurora A (for review see Barr and Gergely, 2007).</p>
          <p id="s3p3">The best studied activator of Aurora A is TPX2, which is required for Aurora A localization to spindles in human cells and Caenorhabditis elegans (Kufer et al., 2002; Ozlu et al., 2005) and binds to and activates Aurora A in vitro (Tsai et al., 2003; Eyers and Maller, 2004). Removal of TPX function in human cells or X. laevis abolishes spindle assembly (Wittmann et al., 2000; Gruss et al., 2001, 2002; Garrett et al., 2002). TPX2 induces microtubule nucleation when added to X. laevis extracts (Gruss et al., 2001), and RNAi of TPX2 in human cells prevents nucleation of microtubules around chromosomes (Tulu et al., 2006), suggesting that chromosome nucleation of microtubules is required for forming a bipolar spindle. However, we do not know if the phenotype of TPX2 RNAi is caused by the lack of chromosome nucleation or by other roles of TPX2. Indeed, we do not know if TPX2 operates in spindle assembly in human cells through activation of Aurora A. In C. elegans, mutants that abolish the interaction between TPX2 and Aurora A result in short bipolar spindles (Ozlu et al., 2005). However, the reason for these short spindles is unknown. In X. laevis extracts, there are conflicting results: in some studies, mutants that remove the N terminus of the TPX2 protein completely prevent the formation of a spindle (Tsai and Zheng, 2005). In others, mutants without the N terminus can support spindle assembly (Brunet et al., 2004), leading to contrasting conclusions of the role of Aurora A activation by TPX2 in spindle assembly. Therefore, the role of TPX2 activation of Aurora A in spindle assembly is unclear (Karsenti, 2005).</p>
          <p id="s3p4">The problem with sorting out the role of TPX2-specific activation of Aurora A in spindle assembly in human cells is that it is difficult to do the correct experiment. This is to introduce point mutants that prevent interaction between the kinase and its activator but maintain the two proteins in the cell. Importantly, the mutant proteins should be expressed under endogenous regulation. Although this is a standard experiment in yeast, to date it has not been possible in human tissue culture cells, which are commonly used to study mitotic processes.</p>
          <p id="s3p5">Bacterial artificial chromosomes (BACs) provide a potential tool to perform these experiments (Kittler et al., 2005). BAC transgenes are likely to be expressed at native or near-native levels and regulated in the same manner as the endogenous gene. This is because they are large (100–200+ kb) pieces of DNA. Because of their size, genes are situated in their native surrounding genomic DNA context and are thus also under the control of any noncoding regulatory elements. All introns are present, allowing for splicing and expression of various splice isoforms. The use of recombineering (Zhang et al., 1998; Muyrers et al., 1999; Copeland et al., 2001) has been reported to allow introduction of mutants into BACs, but these techniques have not been applied to the study of protein function in tissue culture cells. In this paper, we have used BAC-based recombineering to introduce mutants into cells that block the interaction between TPX2 and Aurora A.</p>
          <p id="s3p6">We show that introducing mutations in TPX2 that abolish the interaction between TPX2 and Aurora A result in very short spindles. These spindles are short because spindle poles fail to separate after nuclear envelope breakdown (NEBD). Surprisingly, the spindles are still bipolar and can segregate chromosomes. Examination of microtubule nucleation during spindle formation shows that microtubule nucleation is abolished around chromosomes and kinetochores in these mutant cells. Thus, we conclude that the interaction between TPX2 and Aurora A is required to set the length of the spindle. Our results further suggest that in human cells, the length of the spindle is set by nucleation of microtubules from chromosomes, which interact with microtubules nucleated from centrosomes during spindle formation.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s4h1">Results</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s4s1h1">A BAC transgene containing GFP-tagged mouse TPX2 rescues the phenotype of TPX2 RNAi</h3>
            <div class="content">
              <p id="s4s1p1">To generate a TPX2 transgene with endogenous expression and regulation, we used a BAC containing the mouse TPX2 gene. The mouse gene is resistant to a siRNA directed against human TPX2 yet has 87% similarity to human TPX2 on the protein level. We generated stable clonal lines in human U2OS cells expressing mouse TPX2 protein tagged with a modified “LAP” construct (Cheeseman and Desai, 2005; Poser et al., 2008) that contained an EGFP tag (Fig. S1, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). By generating an antibody to mouse TPX2, we showed that the mouse TPX2-LAP protein appears expressed at similar levels to the endogenous human TPX2 (Fig. 1 a). We call this construct mTPX2WT (see Table I).</p>
              <figure data-origid="fig1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig1">Figure 1.</a>
                <figcaption>
                  <p id="s4s1f1p1">A BAC transgene containing mTPX2-LAP rescues the phenotype of TPX2-RNAi. (a) Western blot of cell lines with anti-mTPX2 antibody, which recognizes both human TPX2 and mTPX2-LAP, and anti–α-tubulin, after TPX2 or CON RNAi. hTPX2 is efficiently depleted in all lines after TPX2 RNAi, whereas mTPX2-LAP transgenes remain. The asterisk represents the position of the TPX2ΔN protein, which runs faster than the full length, as expected. (b) Immunofluorescence of U2OS cells after CON or TPX2 RNAi and TPX2WT cells after TPX2 RNAi, stained for α-tubulin (green), Cep135 (red), and DNA (blue). U2OS cells without a rescuing transgene show a characteristic phenotype after hTPX2 depletion of collapsed poles and lack of a bipolar spindle, whereas TPX2WT cells containing mTPX2-LAP after hTPX2 depletion have normal spindle morphology. Bar,10 μm. (c) Quantification of the TPX2 depletion phenotype and mTPX2-LAP rescue displayed in b. The percent of mitotic cells (n = &gt;1,000 cells per experiment; three to five experiments for each condition) and percent bipolar spindles (n &gt; 50) mitotic cells per experiment; three to five experiments per condition) after CON or TPX2 RNAi in U2OS or TPX2WT cells were determined. Error bars represent standard deviation. The mTPX2-LAP construct is able to rescue depletion of endogenous TPX2.</p>
                </figcaption>
              </figure>
              <figure class="table" data-origid="tbl1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#tbl1">Table I.</a>
                <figcaption>
                  <h5 id="s4s1f2h1">Cell lines used in this study</h5>
                </figcaption>
              </figure>
              <p id="s4s1p2">Upon depletion of TPX2 in U2OS and mTPX2WT cells by RNAi (hTPX2(RNAi)), we observed &gt;95% reduction in TPX2 protein level (Fig. 1 a and Fig. S2, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). Under these conditions in U2OS cells, the spindle poles collapsed near to each other and microtubules did not organize into bipolar structures (Fig. 1 b). By measuring mitotic index and counting the number of bipolar spindles, we showed that the RNAi-resistant mTPX2WT construct successfully rescues TPX2 depletion (Fig. 1, b and c). Western blot and RT-PCR analysis confirmed that in these experiments, when the endogenous human TPX2 is efficiently depleted the mouse TPX2 transgene is still expressed (Figs. 1 a, S2, and S3).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s2h1">The Aurora A–TPX2 interaction is required for Aurora A localization to spindles</h3>
            <div class="content">
              <p id="s4s2p1">We next generated point mutations in the mTPX2WT BAC transgene that, based on previous in vitro studies (Bayliss et al., 2003; Eyers and Maller, 2004; Ozlu et al., 2005), abolish the interaction of TPX2 with Aurora A and the TPX2-dependent activation of Aurora A (Fig. S1). One construct contains three point mutations: Y8A F10A D11A, referred to as mTPX2AAA. The other construct lacks the first 33 aa of the N terminus, referred to as mTPX2ΔN. After hTPX2(RNAi) in stable clonal cell lines expressing the mTPX2AAA or mTPX2ΔN constructs, the endogenous protein was efficiently depleted, whereas the transgene was still expressed (Fig. 1 a). Because of the nature of RNAi rundowns, however, we cannot rule out that a small fraction of human TPX2, not visible by Western blots, may provide some residual TPX2 function.</p>
              <p id="s4s2p2">We first examined whether mTPX2ΔN and mTPX2AAA mutants prevent the interaction of Aurora A with TPX2 in vivo. Untagged cells, cells expressing LAP-tagged mTPX2WT, and cells expressing LAP-tagged mTPX2AAA or mTPX2ΔN were arrested in mitosis with nocodazole, and protein extracts were prepared for immunoprecipitation. Protein extracts were immunoprecipitated using anti-GFP antibodies, and input extract and immunoprecipitated fractions were analyzed by Western blot. As shown in Fig. 2 a, mTPX2WT binds and pulls down Aurora A but does not associate with endogenous hTPX2, suggesting that this interaction is direct. The same immunoprecipitations in untagged cells do not pull down either protein, confirming the specificity of the antibody. In contrast, when TPX2AAA or TPX2ΔN mutants are immunoprecipitated, no detectable Aurora A is coimmunoprecipitated. Thus, these mutants disrupt the TPX2–Aurora A interaction in vivo.</p>
              <figure data-origid="fig2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig2">Figure 2.</a>
                <figcaption>
                  <p id="s4s2f1p1">TPX2 mutants abolish the in vivo interaction between TPX2 and Aurora A as well as Aurora A localization to spindles. (a) Western blots of cell extracts immunoprecipitated to detect interaction of TPX2 with Aurora A. U2OS (untagged), mTPX2WT, mTPX2AAA, and mTPX2ΔN cells were arrested in mitosis with nocodazole, and protein extracts were immunoprecipitated with anti-GFP antibody. Input and immunoprecipitated fractions were run by SDS-PAGE and blotted with either anti-mTPX2 or anti–Aurora A antibody. Anti-GFP antibody pulls down mTPX2WT bound to Aurora A, whereas nothing is immunoprecipitated in the untagged cells. Immunoprecipitating mTPX2WT does not pull down endogenous TPX2. In mTPX2AAA and mTPX2ΔN cells, the mTPX2 mutant transgene is pulled down, but Aurora A is not. The single asterisk represents the position of the TPX2ΔN protein. The double asterisk represents a degradation product of the mTPX2 protein. (b) Immunofluorescence analysis of cell lines after CON or TPX2 RNAi stained for α-tubulin (green), DNA (blue), mTPX2-LAP (anti-GFP; insets), and Aurora A. Aurora A is localized to spindles and centrosomes in U2OS cells (top row). After hTPX2 RNAi, Aurora A is absent from spindles but still on centrosomes (second row). A mouse TPX2-LAP transgene restores Aurora A localization to spindles after hTPX2 RNAi (third row). The same mouse transgene with point mutations introduced to abolish the TPX2–Aurora A interaction no longer is able to recruit Aurora A to spindles after hTPX2 RNAi (bottom row). The pericentrosomal Aurora A localization shown in the mutant similar to that of the TPX2 depletion was observed in images of 12 of 14 cells. In 2 of 14 images, the Aurora A localization in the mutant reflected a more centrosomal localization. Bar, 10 μm.</p>
                </figcaption>
              </figure>
              <p id="s4s2p3">We next examined whether TPX2ΔN and TPX2AAA mutants prevent localization of Aurora A to spindles. We first confirmed previous results that spindle-associated Aurora A is dependent on TPX2 for its localization to spindle microtubules, but not centrosomes, by depleting TPX2 from wild-type cells (Fig. 2 b, second row; Kufer et al., 2002; Ozlu et al., 2005). After hTPX2(RNAi) in mTPX2WT cells, Aurora A is efficiently recruited to spindles, further confirming that the mTPX2-LAP construct rescues wild-type TPX2 function (Fig. 2 b, third row). In contrast, after hTPX2(RNAi) in TPX2AAA (Fig. 2 b, bottom row) or TPX2ΔN (not depicted) mutants, Aurora A is no longer recruited to the spindle but remains on the centrosomes, which is similar to the TPX2 depletion. Thus, we demonstrate that mutants that block the interaction between TPX2 and Aurora A in vitro and in vivo prevent the localization of Aurora A to spindles in vivo.</p>
              <p id="s4s2p4">We next asked whether the interaction with Aurora A is required for TPX2 interaction with spindles. Time-lapse videos showed that mTPX2WT was recruited to asters immediately after NEBD and spread onto the spindle as it formed. (Video S1, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). Analysis of fixed samples immunostained for human TPX2 or mouse TPX2 in metaphase and anaphase showed that TPX2 localizes preferentially to kinetochore fibers (Fig. 3). Indeed, cells with anaphase lagging chromosomes maintained TPX2 staining along microtubules connecting these lagging chromosomes but not on all microtubules (Fig. S4, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). mTPX2WT cells, as well as both the mutant mTPX2AAA and mTPX2ΔN cells, after hTPX2(RNAi) displayed the same dynamic localization of TPX2-LAP through mitosis (Videos S2 and S3) and also retained kinetochore fiber–specific localization upon immunofluorescence with anti–mouse TPX2 antibodies (Fig. 3). Thus, we show that TPX2 localizes to kinetochore fibers and that the interaction between TPX2 and Aurora A is not required for this localization.</p>
              <figure data-origid="fig3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig3">Figure 3.</a>
                <figcaption>
                  <p id="s4s2f2p1">TPX2 localizes to kinetochore fibers independent of interaction with Aurora A. Immunofluorescence analysis of TPX2 localization. (a) U2OS cells in metaphase (top row) and anaphase (bottom row) stained for DNA, CREST, hTPX2 (green), and α-tubulin (black and white). Insets are enlarged to show TPX2 localization to microtubules that end at kinetochores (kinetochore fibers). (b) mTPX2WT cells after hTPX2(RNAi) with same staining as in a, except with mTPX2 instead of hTPX2 to recognize the transgene. (c) mTPX2AAA cells after hTPX2(RNAi) with same staining as in b. mTPX2 mutated to abolish interaction with Aurora A is still able to localize to kinetochore-fibers. Bar, 10 μm.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s3h1">The interaction of TPX2 with Aurora A is required for correct spindle length and spindle pole separation</h3>
            <div class="content">
              <p id="s4s3p1">Analysis of fixed cells after hTPX2(RNAi) in mTPX2AAA or mTPX2ΔN mutant cells showed that spindles were smaller than normal (Fig. 2, bottom left; and Figs. 3 c, and 4 a). To quantify the defect, cells were fixed and stained with DAPI as well as with antibodies against α-tubulin, Cep135 (to mark centrosomes), and CREST (to mark kinetochores). Centrosome-to-kinetochore distances were measured in three-dimensional space. Cells harboring TPX2 mutants had shorter mean centrosome-to-kinetochore distances compared with those of wild type (Fig. 4 b, 2.58 ± 0.48 μm for mTPX2AAA and 2.43 ± 0.62 μm for mTPX2ΔN, compared with 4.35 ± 0.52 μm in wild type).</p>
              <figure data-origid="fig4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig4">Figure 4.</a>
                <figcaption>
                  <p id="s4s3f1p1">The TPX2–Aurora A interaction is required for spindle pole separation and spindle length establishment. (a) Immunofluorescence of mTPX2WT, mTPX2AAA, and mTPX2ΔN mitotic cells after hTPX2(RNAi), stained for CREST, Cep135, DNA, and α-tubulin. Centrosomes are collapsed to chromatin when the TPX2–Aurora A interaction is abolished. Bar, 10 μm. (b) Centrosome-kinetochore distances are shorter in mTPX2AAA and mTPX2ΔN mutants. The centrosome-kinetochore distance from cells stained as in a was determined by measuring individual centrosome-kinetochore distances from metaphase cells in three dimensions (see Materials and methods). For each condition, measurements were taken from four independent cells (eight centrosomes). Red dots show the data points measured and the bars show the means. (c) In TPX2 mutants unable to bind or activate Aurora A, spindle poles collapse immediately after NEBD and only slightly elongate to form shorter metaphase spindles. Distances between spindle poles were measured in three dimensions from live-cell image stacks taken at 1-min intervals of mTPX2WT, mTPX2AAA, and mTPX2ΔN (mTPX2-GFP) cells after hTPX2(RNAi). Thick lines represent means and thin lines represent individual videos. (d) Representative still images from time-lapse recordings quantified in b.</p>
                </figcaption>
              </figure>
              <p id="s4s3p2">To investigate relative centrosome distances as cells are forming spindles, we imaged spindle assembly by time-lapse microscopy in mTPX2AAA or mTPX2ΔN mutant cells after hTPX2(RNAi). In mTPX2WT cells, mTPX2-LAP is recruited to asters after NEBD, and the forming spindle poles then elongate as a bipolar spindle is established with a final metaphase pole-to-pole distance of 12–15 μm (Fig. 4, c and d; and Video S1). When mTPX2AAA or mTPX2ΔN cells were depleted of endogenous TPX2, the mutant protein was still recruited to asters immediately after NEBD but, strikingly, the asters immediately collapsed close to each other and then only slightly elongated to form spindles with poles 7–9 μm apart, (Fig. 4, c and d; and Videos S2 and S3). Collectively, these results show that the interaction between TPX2 and Aurora A is required to establish spindles of the correct length by facilitating the separation of spindle poles after NEBD.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s4h1">TPX2 mutants have compromised fidelity of division</h3>
            <div class="content">
              <p id="s4s4p1">We next looked at the consequences of the mTPX2AAA or mTPX2ΔN mutations on cell division in more detail by monitoring cells completing mitosis after hTPX2(RNAi). Although all wild-type cells observed successfully completed mitosis and segregated DNA, 15% of mTPX2AAA and 35% of mTPX2ΔN cells failed to accurately segregate their DNA into two daughter cells, often dividing into three after anaphase (Fig. 5 a).</p>
              <figure data-origid="fig5">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig5">Figure 5.</a>
                <figcaption>
                  <p id="s4s4f1p1">TPX2 mutants have a higher percentage of defective divisions, delayed mitoses, and increased metaphase BUB1 staining. (a) Percentage of defective mitoses in mTPX2WT, mTPX2AAA, and mTPX2ΔN cells after TPX2 or CON RNAi, quantified by observing time-lapse videos of GFP fluorescence. Actual numbers of cell divisions observed are shown below the histogram. (b) mTPX2AAA and mTPX2ΔN cells after TPX2 RNAi show longer delays in mitosis and a higher percentage of cells exhibiting delays. Lengths of individual mitosis are plotted for mTPX2WT, mTPX2AAA, and mTPX2ΔN cells after TPX2 or CON RNAi. Percentages shown are the fraction of cells showing lengths of mitoses greater than 120 min. (c) Immunofluorescence of mTPX2WT and mTPX2ΔN prometaphase and metaphase cells after TPX2 RNAi, stained for BUB1 or MAD2. Insets show DNA (DAPI) from the same cell at lower magnification. In mTPX2WT cells, MAD2 and BUB1 foci are enriched in prophase but not metaphase. In mTPX2ΔN cells, MAD2 enrichment is gone in metaphase cells, whereas some Bub1-positive foci remain. Bars, 10 μm.</p>
                </figcaption>
              </figure>
              <p id="s4s4p2">We also monitored spindle checkpoint proteins on kinetochores by staining with antibodies against Mad2 or Bub1, two kinetochore-associated checkpoint proteins. (Li and Benezra, 1996; Waters et al., 1998; Skoufias et al., 2001). In mTPX2WT, mTPX2AAA, and mTPX2ΔN cells after hTPX2(RNAi), prometaphase kinetochores display enriched Bub1 and Mad2 staining, showing that the checkpoints are functional (Fig. 5 c and not depicted). The Mad2 enrichment is lost in metaphase mTPX2WT cells, as expected, as well as in the small metaphase spindles established in the mutants, suggesting that the Mad2 checkpoint may be satisfied in these mutants and that kinetochore fibers have attached to kinetochores. However, Bub1 staining in mTPX2AAA and mTPX2ΔN cells after hTPX2(RNAi) revealed that Bub1-enriched kinetochores are occasionally seen in metaphase plates in these cells, unlike in mTPX2WT cells (Fig. 5 c and not depicted). Accordingly, a fraction of mTPX2AAA (20%) and mTPX2ΔN (37.5%) cells show extensive delays (&gt;120 min) in mitosis by time-lapse imaging (Fig. 5 b). Thus, cells in which the TPX2–Aurora A interaction is inhibited are often able to form small bipolar spindles and divide, but these spindles are not sufficient to maintain the overall fidelity of division.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s5h1">The Aurora A–dependent function of TPX2 is required for spindle pole separation but not for bipolarity</h3>
            <div class="content">
              <p id="s4s5p1">Although spindles in mTPX2AAA or mTPX2ΔN mutant cells were shorter, they were still bipolar, satisfied the Mad2 checkpoint, and exited mitosis. Longer exposure to RNAi or use of more concentrated siRNA did not result in failure by these cells to form bipolar spindles (unpublished data). This is in contrast to cells depleted of TPX2, which arrest in mitosis, as well as cells depleted of Aurora A, whose phenotypes vary from failure to enter mitosis to defects in cytokinesis (for review see Barr and Gergely, 2007). To compare spindle dynamics of the TPX2 Aurora A binding mutants to that of wild-type cells depleted of TPX2 or Aurora A, we imaged cells containing a fluorescent α-tubulin reporter after RNAi (Videos S4–S6, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). Beginning ∼16 h after TPX2 RNAi, cells entering mitosis generated normal prophase asters, but at NEBD these asters collapsed together, which is similar to the mTPX2ΔN and mTPX2AAA mutants (Fig. 6, a and b; and Video S5). In contrast to the TPX2 mutants, however, cells depleted of TPX2 where unable to separate spindle poles or establish a DNA metaphase plate and the spindles remained collapsed for several hours, after which the spindle poles fragmented and the cells either exited mitosis without chromosome segregation or apoptosed. We found that cells depleted of Aurora A, although also displaying a collapse of spindle poles after NEBD (Fig. 6, a and b; and Video S6), were often able to generate small bipolar spindles and divide and with similar mean pole separation through mitosis as the mTPX2ΔN and mTPX2AAA mutants. Some Aurora A RNAi cells displayed additional defects such as centrosome fragmentation and cytokinesis defects (unpublished data). Thus, it appears that TPX2 has a second Aurora A–independent role in organizing the spindle microtubules into a bipolar array, as cells depleted of Aurora A, although having multiple defects in mitosis, are more often able to assemble bipolar spindles and divide than TPX2 depleted cells.</p>
              <figure data-origid="fig6">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig6">Figure 6.</a>
                <figcaption>
                  <p id="s4s5f1p1">Depletion of TPX2 from cells results in a spindle defect phenotype distinct from TPX2 mutants or Aurora A depletion. (a) Distances between spindle poles were measured in three dimensions from live-cell image stacks taken at 3-min intervals of U2OS cells stably transfected with α-tubulin–mCherry after treatment with CON, TPX2, or AurA siRNA. Thick lines represent means and thin lines represent individual videos. The dashed line represents data from Fig. 4 for reference. (b) Representative still images from time-lapse recordings quantified in a. The cells shown transfected with CON or AurA siRNAs completed mitosis and divided, whereas the cell depleted of TPX2 arrested in mitosis for several hours and displayed fragmented spindle poles. Bar, 10 μm.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s6h1">The TPX2–Aurora A interaction stabilizes microtubules in prometaphase</h3>
            <div class="content">
              <p id="s4s6p1">The requirement of the TPX2–Aurora A interaction for prometaphase spindle pole separation and the preferential localization of TPX2 to kinetochore fibers prompted us to look at microtubule stability in prometaphase. To assay for microtubule stability, we used cold- and calcium-mediated depolymerization assays. When cells are subjected to cold or calcium treatment for short periods of time, the less stable microtubules are depolymerized initially, leaving preferentially stabilized microtubule populations such as kinetochore fibers intact (Rieder, 1981; Mitchison et al., 1986). When kinetochore fiber stability is compromised, kinetochore fibers also depolymerize in this assay (Sillje et al., 2006).</p>
              <p id="s4s6p2">Immunofluorescence analysis of cold- or calcium-treated mTPX2WT prometaphase cells after hTPX2(RNAi) revealed stable subpopulations of microtubules, which were long and had a bipolar spindle–like shape but were not kinetochore fibers per se, as these microtubules were not yet associated with kinetochores in these early stages of mitosis. (Fig. 7 and Fig. S5, available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1). After treating prometaphase mTPX2ΔN and mTPX2AAA cells after hTPX2(RNAi) with cold or calcium, there was a reduction in the number, length, and organization of these preferentially stabilized microtubules. Surprisingly, when we looked at microtubule stabilization in metaphase mTPX2ΔN or mTPX2AAA cells after hTPX2(RNAi), there was no observable defect in the stability of metaphase kinetochore fibers, even though the kinetochore fibers and spindles were shorter (Fig. 7, right; and Fig. S5, right). Thus, it appears that Aurora A activation by TPX2 is required for early stability and organization of microtubules in prometaphase.</p>
              <figure data-origid="fig7">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig7">Figure 7.</a>
                <figcaption>
                  <p id="s4s6f1p1">The TPX2–Aurora A interaction is required for cold-stable microtubule stability and organization in prometaphase. mTPX2WT, mTPX2AAA, and mTPX2ΔN cells were transfected with TPX2 or CON siRNA. 48 h after transfection, cells were treated on ice for 5 min and fixed. Fluorescence images are shown of cells stained with anti–α-tubulin (microtubules) and DAPI (DNA) showing cold-stable microtubule populations. Bar, 10 μm.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s4s7h1">The TPX2–Aurora A interaction is required for chromatin/kinetochore-mediated microtubule nucleation</h3>
            <div class="content">
              <p id="s4s7p1">The microtubules that contribute to the forming prometaphase spindle have been proposed to originate from both centrosomes and chromosomes. TPX2 has previously been shown to be required for kinetochore/chromatin-dependent microtubule nucleation in human cells (Tulu et al., 2006). To test whether TPX2-dependent Aurora A activation was required for kinetochore/chromatin nucleation, we used a microtubule repolymerization assay to distinguish nucleation of microtubules from centrosomes and chromatin. In this assay, U2OS cells were transferred to an ice bath for 30 min to completely depolymerize microtubules. Cells were then transferred in brief to 37°C media to allow for repolymerization and immediately fixed and stained for tubulin, DNA, Cep135, and CREST. After 90 s of incubation at 37°C, wild-type U2OS cells displayed characteristic microtubule polymerization from centrosomes, as well as polymerization clearly associated with chromatin and kinetochores and away from centrosomes (Fig. 8). After hTPX2(RNAi) in U2OS cells, microtubule polymerization around chromatin and kinetochores was abolished, whereas centrosome microtubule nucleation remained, confirming previous observations (Tulu et al., 2006). After hTPX2(RNAi) in mTPX2WT cells, microtubule polymerization was again evident around both centrosomes and chromatin/kinetochores in a majority of cells. mTPX2 protein also colocalized with both microtubule populations (unpublished data). In contrast, polymerization was rarely seen around chromatin and kinetochores after hTPX2(RNAi) in mTPX2ΔN or mTPX2AAA mutant cells (Fig. 8). Thus, the TPX2–Aurora A interaction is required for chromatin/kinetochore-dependent microtubule nucleation.</p>
              <figure data-origid="fig8">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483532/#fig8">Figure 8.</a>
                <figcaption>
                  <p id="s4s7f1p1">The TPX2–Aurora A interaction is required for chromatin/kinetochore-mediated microtubule nucleation. U2OS, mTPX2WT, mTPX2AAA, and mTPX2ΔN cells transfected with TPX2 or CON siRNA were treated on ice to completely depolymerize microtubules, transferred to 37°C for 90 s to allow repolymerization, and immediately fixed. Immunofluorescence images are shown of cells stained with α-tubulin and CREST (top rows) and Cep135 and DNA (bottom rows). In addition to aster microtubule polymerization associated with centrosomes, microtubule polymerization broadly associated with chromatin and often specifically associated with kinetochores was evident in all cell lines after CON siRNA and in mTPX2WT cells after TPX2 RNAi. In U2OS, mTPX2AAA, and mTPX2ΔN cells after TPX2 RNAi, microtubule repolymerization was no longer seen associated with chromatin or kinetochores, but centrosome-associated microtubule aster polymerization was still evident. Bar, 10 μm.</p>
                </figcaption>
              </figure>
            </div>
          </section>
        </div>
      </section>
      <section data-type="">
        <h2 id="s5h1">Discussion</h2>
        <div class="content">
          <p id="s5p1">To be able to study TPX2-dependent Aurora A activation, we needed a system in which we could accurately infer function from a mutated TPX2 transgene. Genetic analysis of protein function in human cells has traditionally relied on the expression of cDNAs or cDNA gene fragments, sometimes transiently and often overexpressed, to infer function. However, studies in human cells using cDNA-based TPX2 transgenes had found that defects in mitosis arise from its expression in this form in the cell (Gruss et al., 2002; Ciciarello et al., 2004; Stewart and Fang, 2005). The increasing use of structure to predict mutants that block interaction between proteins will require general methods to introduce these mutants into cells in a controlled manner. When analyzing mutants for phenotypes proper regulation is critical, as overexpression or misregulation of a transgene could mask or induce mutant phenotypes, even when a wild-type rescue construct displays no altered phenotype. In this paper, we define a system to bring genetic analysis in mammalian cells closer to the accepted standard of yeast genetics. We show that an RNAi-resistant mTPX2-LAP transgene efficiently rescues the depletion of the endogenous protein, setting a basis for controlled structure-function studies in mammalian cells.</p>
          <p id="s5p2">The principle role that we have found for the interaction between TPX2 and Aurora A is to build spindles of the correct length. Surprisingly, mutant spindles are still bipolar and can segregate chromosomes, although these shorter spindles have a higher incidence of mitotic failure. Thus, this phenotype allows us to study factors specifically required for setting spindle length. How does TPX2-dependent activation of Aurora A establish the distance between two spindle poles? The most obvious defect we have seen is that chromatin/kinetochore-based nucleation fails when TPX2 and Aurora A do not interact. This result reveals two things about chromatin nucleation. First, it is not only present but required for correct spindle assembly in normal human cell mitosis. Second, its primary role is likely to be in determining the shape and size of the spindle. A bipolar spindle can still form without TPX2-mediated Aurora A activation and chromatin/kinetochore-based microtubule nucleation.</p>
          <p id="s5p3">A second consequence of the failure of TPX2 to bind Aurora A is that forming spindle microtubules are less stable during prometaphase but not metaphase, suggesting that determinants of spindle length establishment act early in prometaphase soon after NEBD. We know little about how microtubules are stabilized during prometaphase or which microtubule populations participate. From our analysis, it appears that a hyperstable “skeleton” of a bipolar spindle, which is evident in depolymerization assays, may be formed through the selective stabilization of microtubules by chromosomes. We speculate that as microtubules from the prometaphase asters and the chromatin overlap, these overlapping microtubules are selectively stabilized to form the developing spindle. Kinetochore fibers then form within this developing spindle. In this model, the degree of chromatin nucleation would determine the position of the centrosomes with respect to the chromosomes. Therefore, spindle poles will tend to move to the chromosomes after NEBD in the absence of chromatin nucleation. The activity of TPX2 is thought to require the activity of the small GTPase Ran, which releases TPX2 from importins (Gruss et al., 2001; Schatz et al., 2003; Trieselmann et al., 2003; Tsai et al., 2003). Because chromosomes activate Ran through RCC1-mediated GTP exchange (Bischoff and Ponstingl, 1991; Nemergut et al., 2001), the chromosomes may therefore be activating Aurora A in their vicinity through TPX2 release from importin (Tsai et al., 2003). Indeed, Aurora A beads stimulate microtubule nucleation in their vicinity in X. laevis extracts (Tsai and Zheng, 2005). Thus, it seems likely that chromosomes activate Aurora A by release of TPX2 from importin. The amount of Aurora A activated around chromosomes would determine either the amount of nucleation or, more likely given the localization of TPX2 to kinetochore microtubules, would stabilize nucleated microtubules, allowing them to incorporate in the forming spindle. Of course, we cannot rule out that TPX2 activation of Aurora A contributes to spindle length by activating other pathways, such as modulation of astral microtubule attachment to the cortex (Rosenblatt et al., 2004).</p>
          <p id="s5p4">Analysis of TPX2 mutants unable to bind Aurora A suggests independent functions of TPX2 in spindle assembly. The Aurora A–independent second role of TPX2 appears to be required to organize microtubules into bipolar spindles and is likely dependent on the C terminus of the protein. Indeed, even in cells depleted of Aurora A, unlike TPX2-depleted cells, a majority of cells are able to form bipolar spindles and divide, although it is possible that Aurora A activity in this case is not completely inhibited. TPX2 protein mutated to abolish the Aurora A interaction still binds microtubules in vivo, as its spindle localization is not changed from that of the wild-type. This is consistent with previous studies in humans and X. laevis, which have found that the microtubule binding (Schatz et al., 2003; Trieselmann et al., 2003) and bundling (Schatz et al., 2003) properties of TPX2 are independent of the Ran–Aurora A activation pathway (Schatz et al., 2003; Trieselmann et al., 2003). A recent study by Manning and Compton (2007) hypothesizes that TPX2 serves a function as a microtubule cross-linker in spindle assembly. The TPX2 C terminus may facilitate bipolarity through a microtubule bundling activity that is necessary for kinetochore fiber establishment and stabilization of the bipolar array. This would explain why in a TPX2 depletion, kinetochore fibers or bipolar spindles do not correctly form, whereas in TPX2 mutants unable to interact with Aurora A, kinetochore-fibers are still present and stable (although shorter) in metaphase bipolar spindles and mutant TPX2 is still able to localize to these kinetochore fibers. In support of this idea of a separate function for the C terminus, a recent study by Eckerdt et al. (2008) found that in X. laevis, the C-terminal domain of TPX2 interacts with Eg5, a motor protein required for spindle bipolarity.</p>
          <p id="s5p5">It is interesting to compare our work in somatic cells with that in X. laevis meiotic extracts, where the role of TPX2–Aurora A interaction in spindle assembly has also been studied. Our results show that although the N terminus of TPX2 is not required to assemble a bipolar spindle, it is required for proper spindle length. In cycled X. laevis extracts (Brunet et al., 2004), TPX2 protein lacking an N-terminal region can support bipolar spindle assembly, as we saw in our mutant cells, and spindle length was not measured. However, in these X. laevis extracts, this TPX2 construct can support spindle assembly around DNA-coated beads, where there are no centrosomes. Because spindle assembly around DNA-coated beads is thought to require chromatin nucleation, this is difficult to reconcile with our finding that chromatin nucleation is not seen in mutants that block the interaction of TPX2 with Aurora A. One possibility is that there are alternative pathways in X. laevis–cycled spindles that can support chromatin-mediated nucleation. The C terminus of TPX2 would then be sufficient to support bipolarity.</p>
          <p id="s5p6">Interestingly, another study showed that in cytostatic factor–arrested spindles, TPX2 lacking the N-terminal Aurora A binding region is not able to support spindle assembly at all (Tsai and Zheng, 2005). The difference between cytostatic factor extracts used in this study and the cycled extracts from Brunet et al. (2004) could be the degree to which chromatin-mediated nucleation is dependent on Aurora A. In cytostatic factor–arrested spindles, bipolar spindle assembly may be completely dependent on chromatin-mediated microtubule nucleation (through Aurora A activation), whereas in human cells centrosomal microtubule nucleation is able to support bipolar spindle assembly when chromatin-mediated microtubule nucleation through the TPX2–Aurora A activation is abolished.</p>
          <p id="s5p7">In C. elegans embryos, depletion of the TPX2 homologue TPXL-1 has a similar phenotype to mutating the Aurora A–interacting region of TPX2 in either C. elegans or human cells (Ozlu et al., 2005): as spindles form, poles collapse together but are able to rescue to form bipolar spindles and divide. Interestingly, TPXL-1 lacks the homologous C terminus that is required for bipolarity in human cells. Thus, it appears that although the Aurora A–activating function of TPX2 is conserved, other proteins or processes in C. elegans provide the Aurora A–independent function observed in human cells. Alternatively, this function may not be required for spindle assembly in C. elegans, perhaps because the kinetochores are holocentric.</p>
          <p id="s5p8">To understand why Aurora A localization to spindles is required to set spindle length, it will be necessary to identify the target of TPX2-mediated Aurora A activation. There are several intriguing candidates for this target. For example, NEDD1 (GCP-WD) localizes to mitotic spindles and is required for chromatin-dependent microtubule nucleation (Luders et al., 2006). Furthermore, mutation of a phosphorylation site in NEDD1 abolishes its spindle localization as well as spindle assembly. Two proteins that localize to mitotic spindles and are known to be phosphorylated by Aurora A are HURP and Eg5 (Giet et al., 1999; Koffa et al., 2006; Wong et al., 2008). HURP localizes preferentially to regions of kinetochore fibers in the vicinity of chromosomes, is also required for chromatin-dependent microtubule nucleation, and has a role in increasing kinetochore fiber stability (Giet et al., 1999; Koffa et al., 2006; Sillje et al., 2006; Wong and Fang, 2006). Eg5 is thought to function as a plus-end–directed motor that binds overlapping microtubules and pushes the centrosomes apart (Sawin et al., 1992; Kapitein et al., 2005). Both of these proteins (HURP and Eg5) were recently found in a complex with TPX2 and Aurora A in X. laevis extracts, and this complex has been proposed to be responsible for Aurora A–dependent bipolar spindle assembly (Koffa et al., 2006; for review see Barr and Gergely, 2007). There is, however, no evidence to date that the same complex exists in human cells, and we have not in our own hands detected interaction between these proteins from TPX2 or Eg5 immunoprecipitations in human cells (unpublished results). Preliminary studies in our laboratory show that NEDD1 and HURP localizations are not affected in TPX2ΔN or TPX2AAA mutants (unpublished data), although this does not rule out that their activity may be affected by TPX2-dependent Aurora A phosphorylation.</p>
          <p id="s5p9">To study the complex roles of kinases in mammalian cells accurately will require the separation and identification of individual specific functions of these kinases. In this paper, we have identified the role of TPX2-dependent Aurora A activation as required for chromatin-mediated microtubule nucleation and spindle pole separation. Using BACs as a source of RNAi-resistant transgenes for detailed structure-function studies will be applicable to all areas of molecular biology.</p>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s6h1">Materials and methods</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s6s1h1">Cloning and plasmids</h3>
            <div class="content">
              <p id="s6s1p1">The BAC RP24-370E11 containing mTPX2 was purchased from BACPAC Resources Center. A LAP tag cassette (Poser et al., 2008) was recombined at the C terminus of mTPX2 by Red E/T-based recombination (Zhang et al., 1998; Muyrers et al., 2001). Point mutations were introduced through recombineering using counterselection based on an RpsL-amp cassette (Wang et al., 2006) as described in Counter Selection BAC Modification kit (Gene Bridges). α-Tubulin–mCherry plasmid was a gift of J. Ellenberg (European Molecular Biology Laboratory, Heidelberg, Germany).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s2h1">Antibodies</h3>
            <div class="content">
              <p id="s6s2p1">Antibodies against human Cep135 (NP_079285) were generated in rabbits using a purified GST fusion protein of aa 695–838. Antibodies against mouse TPX2 (NP_082385) were generated in rabbits using purified GST fusion proteins of aa 529–744 and 648–744. For affinity purification, the same regions were cloned in frame to MPB in the pMAL-c2 vector and the fusion proteins purified essentially as suggested by the manufacturer (New England Biolabs, Inc.). The MBP fusion proteins were coupled to 1 ml NHS HiTrap columns (GE Healthcare) and affinity purification was performed using standard procedure. For immunoprecipitations, polyclonal goat anti-GFP antibodies were used (Poser et al., 2008).</p>
              <p id="s6s2p2">The following antibodies were obtained from commercial sources: rabbit anti-hTPX2(NB 500–179; Novus Biologicals), human nuclear antibodies to nuclear antigens–centromere autoantibody (CREST; CS1058; Europa Bioproducts), rabbit anti–Aurora A (ab 12875; Abcam), mouse anti-GFP (for Western Blots and immunofluorescence; 7.1 and 13.1; Roche), mouse anti–α-tubulin (DM1a; Sigma-Aldrich), rat anti–α-tubulin (AbD Serotec), rabbit anti-MAD2 (PRB-452C; Covance), mouse anti-Bub1 (IQ255; ImmuQuest). Secondary antibodies used were the following: donkey anti–mouse, –rabbit, or –rat conjugated to Alexa 488, 594, or 647 (Invitrogen) and donkey anti–human conjugated to Texas Red or Cy5 (Jackson ImmunoResearch Laboratories)</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s3h1">Cell culture</h3>
            <div class="content">
              <p id="s6s3p1">U2OS cells were grown in DME containing 10% FBS, 2 mM l-glutamine, 100 U/ml penicillin, and 100 μg/ml streptomycin at 37°C in 5% CO2. BAC constructs and α-tubulin–mCherry were transfected into cells in 6-cm dishes with 20 μl Effectene (Qiagen), according to the manufacturer's protocol, stable line populations were selected on G418, and individual clones were isolated.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s4h1">RNAi</h3>
            <div class="content">
              <p id="s6s4p1">siRNAs against TPX2 (GGGCAAAACUCCUUUGAGA) and Aurora A (AurA; GAACUUCAGAAACUUUCAA) were purchased from Ambion. CON siRNA (control) was also purchased from Ambion (Silencer Negative Control #1 siRNA). siRNAs were transfected into U2OS cells using oligofectamine (Invitrogen). For transfections in 24-well plates for immunofluorescence or westerns, 35,000 cells were added to prewarmed media, and transfection complexes containing 2.5 μl oligofectamine and 40 pmol RNA were added immediately afterward. For transfections for live imaging, 30,000 cells were added to prewarmed media, and transfection complexes containing 1.5 μl oligofectamine and 50 pmol TPX2 or 20 pmol AurA RNA were added immediately afterward. For all transfections, media was changed after 6–8 h.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s5h1">Immunofluorescence</h3>
            <div class="content">
              <p id="s6s5p1">Cells on coverslips were fixed in −20°C methanol unless otherwise stated. Cells were blocked with 0.2% fish skin gelatin (Sigma-Aldrich) in PBS. Cells were incubated with primary antibodies in 0.2% fish skin gelatin in PBS for 20 min at 37°, washed, and then repeated with secondary antibodies. Coverslips were mounted with ProLong Gold with DAPI (Invitrogen) overnight and sealed.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s6h1">Immunoprecipitations</h3>
            <div class="content">
              <p id="s6s6p1">Cells were grown to ∼70% confluency, and 100 ng/ml nocodazole was added for 20–24 h. Mitotic cells were then harvested by shakeoff. Cells were lysed by resuspending in lysis buffer containing 150 mM NaCl, 0.25% NP-40, 50 mM Tris-CL, pH 7.5, 5 mM EDTA, 10% glycerol, and protease inhibitors. Benzonase (EMD) was added to extracts, extracts were centrifuged at 20,000 g for 30 min, and the supernatant was taken as “input”. Extract input was added to goat anti-GFP antibody coupled to Protein G Sepharose beads (GE Healthcare) that were previously preeluted with glycine, pH 2. Beads and extract were rotated for 2 h at 4°C. Beads were washed six times with lysis buffer containing 0.05% NP-40. Proteins were eluted off the beads by adding 0.1 M glycine, pH 2, and analyzed by Western blotting.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s7h1">Western blotting</h3>
            <div class="content">
              <p id="s6s7p1">Cells after RNAi were trypsinized, washed, and resuspended in hot Laemmli buffer. Samples were separated by SDS-PAGE and transferred to a nitrocellulose membrane. The membrane was cut in half and blotted with rabbit anti-mTPX2 and mouse anti–α-tubulin.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s8h1">RT-PCR</h3>
            <div class="content">
              <p id="s6s8p1">Total RNA was isolated from cells after RNAi using the RNeasy kit (Qiagen). cDNA was synthesized with Superscript II reverse transcription (Invitrogen). RNA expression levels were detected by PCR using primers GACACCATTAACCTGCC and GATGACGGTGTTTGGAC, which anneal to both the human and mouse TPX2 gene. Digestion of the product with Hpy188i digested the human-specific PCR product into two ∼100-bp products, whereas the mouse remained undigested. β-Actin signal was amplified using the primers GCAACCGCGAGAAGATGACC and CTCCTTAATGTCACGCACGATTTC.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s9h1">Microscopy and image quantification</h3>
            <div class="content">
              <p id="s6s9p1">All fixed and live images were acquired using a DeltaVision RT imaging system (Applied Precision, LLC; IX70/71 [Olympus]) equipped with a charge-coupled device camera (CoolSNAP HQ; Roper Scientific). Fixed images were acquired in 0.2-μm serial Z sections using a 100× 1.35 NA UPLanApo objective at room temperature. Live cell images were acquired in 2.5- or 1.5-μm serial Z sections at 1- or 3-min intervals using a 60× 1.42 NA PlanApo N objective at 37°C. For live cell imaging, cells were incubated in CO2-independent medium (Invitrogen). Datasets were deconvolved using SoftWoRx (Applied Precision, LLC) software. Distances between spindle poles in live time-lapse images were measured manually in three dimensions using SoftWoRx software. For fixed cell analysis of kinetochore to centrosome distances, deconvolved datasets were reconstructed in three dimensions using Imaris image analysis software (Bitplane). All kinetochore and centrosome x-y-z coordinates were identified for each image and the corresponding distances between them determined.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s10h1">Microtubule regrowth and microtubule stability assays</h3>
            <div class="content">
              <p id="s6s10p1">For the microtubule regrowth assay, 30 or 48 h after transfection with TPX2 or CON siRNA cells on coverslips were incubated with media containing 100 ng/ml nocodazole for 6 h. Coverslips were then transferred to ice-cold media supplemented with 10 mM Hepes, pH 7.25, for 30 min to fully depolymerize microtubules. Cells were then transferred to the same medium at 37°C for 90 s to allow microtubule regrowth and fixed immediately in cold methanol. For the cold-stable microtubule assay, 48 h after transfection with TPX2 or CON siRNAs cells on coverslips were transferred to ice-cold L-15 media (Invitrogen) for 5 min and fixed with 4% formaldehyde. For the calcium-stable microtubule assay, 48 h after transfection with TPX2 or CON siRNAs cells on coverslips were incubated in 0.1 M Pipes, pH 6.8, 1 mM MgCl2, 1 mM CaCl2, and 0.2% Triton X-100 for 2.5 min. Cells were then fixed in the same buffer containing 4% formaldehyde.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s11h1">Online supplemental material</h3>
            <div class="content">
              <p id="s6s11p1">Fig. S1 shows a diagram of BAC modifications present in mTPX2WT, mTPX2AAA, and mTPX2ΔN constructs. Fig. S2 shows that TPX2 RNAi reduces TPX2 protein expression to levels &lt;5% of those of endogenous. Fig. S3 shows that mouse TPX2 is efficiently expressed in cells after RNAi of human TPX2. Fig. S4 shows that TPX2 colocalizes with kinetochore fibers connecting lagging chromosomes. Fig. S5 shows that the TPX2–Aurora A interaction is required for calcium-stable microtubule stability and organization in prometaphase. Video 1 shows that mTPX2-LAP rescues hTPX2(RNAi) and displays localization similar to that of hTPX2. Video 2 shows that mTPX2AAA is recruited to spindles and results in collapse of spindle poles and shorter spindle lengths. Video 3 shows that mTPX2ΔN is recruited to spindles and results in collapse of spindle poles and shorter spindle lengths. Video 4 shows α-tubulin localization and spindle dynamics through mitosis in wild-type cells. Video 5 shows α-tubulin localization and spindle dynamics through mitosis in cells depleted of TPX2. Video 6 shows α-tubulin localization and spindle dynamics through mitosis in cells depleted of Aurora A. Online supplemental material is available at http://www.jcb.org/cgi/content/full/jcb.200802005/DC1.</p>
            </div>
          </section>
        </div>
      </section>
    </article>
  </body>
</html>
