<!DOCTYPE html >
<html id="aKvSBF.Dnx7F2qi8oCJCRPrkQvZK-PMC3596250" data-origid="PMC3596250" class="anndoc" data-anndoc-version="2.0" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="com.jmcejuela.anndoc.v2.bio.NcbiJournalArticleParser_v0_4$"/>
    <meta name="dcterms.source" content="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/"/>
    <title>aKvSBF.Dnx7F2qi8oCJCRPrkQvZK-PMC3596250</title>
  </head>
  <body>
    <article>
      <section data-type="title">
        <h2 id="s1h1">GOLPH3L antagonizes GOLPH3 to determine Golgi morphology</h2>
      </section>
      <section data-type="abstract-precis">
        <h2 id="s2h1">Abstract (precis)</h2>
        <div class="content">
          <p id="s2p1">GOLPH3 is a ubiquitous PI4P effector, critical for Golgi function, and also an oncogene. GOLPH3L, a paralogue in vertebrates, also binds PI4P and localizes to the Golgi, but its expression is restricted to secretory cells. Despite some similarities to GOLPH3, GOLPH3L fails to interact with myosin 18A and functions at the Golgi to antagonize GOLPH3.</p>
        </div>
      </section>
      <section data-type="abstract">
        <h2 id="s3h1">Abstract</h2>
        <div class="content">
          <p id="s3p1">GOLPH3 is a phosphatidylinositol-4-phosphate (PI4P) effector that plays an important role in maintaining Golgi architecture and anterograde trafficking. GOLPH3 does so through its ability to link trans-Golgi membranes to F-actin via its interaction with myosin 18A (MYO18A). GOLPH3 also is known to be an oncogene commonly amplified in human cancers. GOLPH3L is a GOLPH3 paralogue found in all vertebrate genomes, although previously it was largely uncharacterized. Here we demonstrate that although GOLPH3 is ubiquitously expressed in mammalian cells, GOLPH3L is present in only a subset of tissues and cell types, particularly secretory tissues. We show that, like GOLPH3, GOLPH3L binds to PI4P, localizes to the Golgi as a consequence of its PI4P binding, and is required for efficient anterograde trafficking. Surprisingly, however, we find that perturbations of GOLPH3L expression produce effects on Golgi morphology that are opposite to those of GOLPH3 and MYO18A. GOLPH3L differs critically from GOLPH3 in that it is largely unable to bind to MYO18A. Our data demonstrate that despite their similarities, unexpectedly, GOLPH3L antagonizes GOLPH3/MYO18A at the Golgi.</p>
        </div>
      </section>
      <section data-type="">
        <h2 id="s4h1">INTRODUCTION</h2>
        <div class="content">
          <p id="s4p1">The lipid phosphatidylinositol-4-phosphate (PI4P) is essential for secretion via the Golgi (for review, see De Matteis et al., 2005; Balla and Balla, 2006; D'Angelo et al., 2008; Graham and Burd, 2011). GOLPH3 (also referred to in the literature as Vps74p, GPP34, MIDAS, or GMx33α) is a PI4P effector at the Golgi (Wu et al., 2000; Bell et al., 2001; Nakashima-Kamimura et al., 2005; Snyder et al., 2006; Dippold et al., 2009; Wood et al., 2009). As a consequence of binding to PI4P (Dippold et al., 2009) GOLPH3 localizes to the trans-Golgi (Wu et al., 2000; Snyder et al., 2006), the site of enrichment of PI4P (Nakagawa et al., 1996; Wong et al., 1997; Wang et al., 2003; Godi et al., 2004). GOLPH3 acts by providing a link to the actin cytoskeleton through its interaction with myosin 18A (MYO18A; Dippold et al., 2009; Bishé et al., 2012). The GOLPH3/MYO18A complex applies a tensile force to the trans-Golgi membrane, participating in the process of vesicle budding for anterograde trafficking. This tensile force also has the consequence of stretching the Golgi ribbon around the nucleus and flattening the Golgi cisternae, giving the Golgi its characteristic appearance by fluorescence and electron microscopy. Unexpectedly, unbiased screening led to the discovery that GOLPH3 is also an oncogene that is commonly amplified in human cancers (Scott et al., 2009; Kunigou et al., 2011; Li et al., 2011, 2012; Hua et al., 2012; Wang et al., 2012; Zeng et al., 2012; Zhou et al., 2012; Hu et al., 2013).</p>
          <p id="s4p2">In the budding yeast Saccharomyces cerevisiae the GOLPH3 homologue Vps74p also localizes to the Golgi by binding to PI4P (Dippold et al., 2009; Wood et al., 2009). Budding yeast lack a canonical MYO18A, and it is uncertain whether Vps74p might bind to one of the smaller number of primordial myosins that are found in yeast. Of note, the observation that the type V myosin, Myo2p, depends on PI4P through an unknown adaptor for its role in secretion (Santiago-Tirado et al., 2011) raises the possibility that Vps74p might function in that capacity.</p>
          <p id="s4p3">Yeast strains in which VPS74 has been deleted display mislocalization of several mannosyltransferases that are unique to yeast (Tu et al., 2008, 2012; Schmitz et al., 2008). However, recent evidence suggests the effect to be indirect, as a consequence of compensatory changes in lipid levels in the vps74Δ yeast strains (Wood et al., 2012). Indeed, mutations that phenocopy the changes in lipid levels also produce the same mislocalization of the mannosyltransferases. Of interest, the yeast VPS74 gene is arranged in a head-to-head configuration with the FRQ1 gene, which encodes a heterodimeric partner required for the function of the Pik1p PI-4-kinase (Hendricks et al., 1999; Huttner et al., 2003; Strahl et al., 2007). It is unknown whether unintended misregulation of the PI-4-kinase in the vps74Δ yeast may be complicating the interpretation of the knockout phenotypes. The observation that human GOLPH3 can partially rescue the defect in vps74Δ yeast argues that they share an evolutionarily conserved function (Tu et al., 2008). However, questions clearly remain as to the nature of that conserved function.</p>
          <p id="s4p4">Vertebrate genomes encode a paralogue of GOLPH3 called GOLPH3L (also referred to in the literature as GMx33β or GPP34R; Wu et al., 2000; Bell et al., 2001). Relatively little is known about GOLPH3L. To provide more insight into the function of the GOLPH3 family, we set out to study the function of GOLPH3L. Initially we expected that GOLPH3L might function similarly to GOLPH3 since vertebrate gene family members often segregate their functions by cell type or exhibit overlapping, complementary functions (Demuth et al., 2006). Surprisingly, our results demonstrate that GOLPH3L differs significantly from GOLPH3, opposing the action of GOLPH3/MYO18A in maintaining Golgi morphology.</p>
        </div>
      </section>
      <section data-type="results">
        <h2 id="s5h1">RESULTS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s5s1h1">Sequence comparison between GOLPH3 and GOLPH3L</h3>
            <div class="content">
              <p id="s5s1p1">Although the genomes of lower organisms encode only a single GOLPH3 protein, vertebrate genomes contain two homologous genes encoding GOLPH3 and GOLPH3L. We compared the protein sequences of GOLPH3 and GOLPH3L from several organisms. Multiple sequence alignment (Figure 1, A and B, and Supplemental Figure S1) of GOLPH3 and GOLPH3L from several vertebrate species, as well as from several invertebrate species that only possess GOLPH3, reveals high conservation of the C-terminus in all members of the family. This conserved region corresponds to the minimal portion of GOLPH3 that is required for the ability to bind PI4P and localize to the Golgi (Dippold et al., 2009). The observation that GOLPH3 and GOLPH3L are highly conserved and are equally distant from the primordial GOLPH3 found in lower organisms initially led us to consider whether GOLPH3 and GOLPH3L might function similarly.</p>
              <figure data-origid="F1">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F1">FIGURE 1:</a>
                <figcaption>
                  <p id="s5s1f1p1">(A) Comparison of human GOLPH3 and GOLPH3L. Amino acid identity and similarity for entire protein, N-terminal domain, and C-terminal PI4P-binding domain are indicated. (B) Dendrogram of GOLPH3 and GOLPH3L family members created using data from the multiple sequence alignment in Supplemental Figure S1. Distances represent degree of divergence of amino acid sequence.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s2h1">Secretory tissues express GOLPH3L</h3>
            <div class="content">
              <p id="s5s2p1">Because mammalian gene family members often serve related, cell-type–specific functions, we sought to determine the relative expression of GOLPH3 and GOLPH3L in various cell lines and tissue types. First, we tested whether the antibody that we had raised against GOLPH3 was equally capable of detecting GOLPH3 and GOLPH3L by Western blot. We transfected HeLa cells with expression vectors for enhanced green fluorescent protein (EGFP)–GOLPH3 or EGFP-GOLPH3L fusion proteins. Western blotting using an antibody to EGFP demonstrated approximately equal expression of the fusion proteins (Figure 2A). When we probed the Western blots with our anti-GOLPH3 antibody we detected bands of similar intensity, thus demonstrating that in Western blot assays our antibody recognizes GOLPH3 and GOLPH3L equally well.</p>
              <figure data-origid="F2">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F2">FIGURE 2:</a>
                <figcaption>
                  <p id="s5s2f1p1">Detection of endogenous GOLPH3 and GOLPH3L. (A) Pan-specific GOLPH3 antibody detects GOLPH3 and GOLPH3L equally well. EGFP-GOLPH3 (G3) and EGFP-GOLPH3L (G3L) expressed at similar levels in HeLa cells, as shown by EGFP Western blot, are detected equally well by antibody raised against GOLPH3, as shown by GOLPH3 Western blot. (B) Endogenous GOLPH3 and GOLPH3L are detected by pan-specific GOLPH3 antibody. SDS lysates from HEK 293 cells, each sample transfected with a combination of two of the following siRNA oligos: control siRNA, siRNA specific to GOLPH3, and each of three siRNAs specific to GOLPH3L. All samples were Western blotted using pan-specific GOLPH3 antibody and GAPDH antibody as a loading control. Lysates from control siRNA–treated cells are loaded at different relative amounts to provide a standard curve to allow semiquantitative assessment of knockdown efficiency. The upper band detected by pan-specific GOLPH3 antibody (∼34 kDa) corresponds to the predicted molecular weight of GOLPH3 and is specifically depleted when cells are transfected with siRNA specific to GOLPH3. The lower band (∼33 kDa) corresponds to the predicted molecular weight of GOLPH3L and is specifically depleted when cells are transfected with each of three siRNAs specific to GOLPH3L. Combined transfection of siRNAs to GOLPH3 and GOLPH3L results in depletion of both proteins.</p>
                </figcaption>
              </figure>
              <p id="s5s2p2">Next we tested our ability to distinguish endogenous GOLPH3 and GOLPH3L by Western blotting by virtue of the fact that GOLPH3 is slightly larger than GOLPH3L. We transfected HEK 293 cells with each of three specific small interfering RNAs (siRNAs) designed to knock down GOLPH3L, control siRNAs, or an siRNA that we previously validated as specific for GOLPH3 (Dippold et al., 2009). We examined SDS lysates by Western blot, as shown in Figure 2B. In HEK 293 cells we can resolve two bands in the expected range of 33–34 kDa. The intensity of the upper band is reduced upon knockdown of GOLPH3, whereas the intensity of the lower band is reduced upon knockdown of GOLPH3L. We conclude that the upper and lower bands represent GOLPH3 and GOLPH3L, respectively.</p>
              <p id="s5s2p3">We used Western blotting with our pan-GOLPH3 antibody to examine the relative expression of GOLPH3 and GOLPH3L in several mammalian cell lines of various origins (Figure 3A). Western blotting of glyceraldehyde-3-phosphate dehydrogenase (GAPDH) demonstrates similar loading of total cellular protein for each cell line. In HeLa cells GOLPH3 is an abundant protein (∼500,000 molecules per cell; Dippold et al., 2009). We observe a similarly high level of expression of GOLPH3 in all of the cell lines examined, including cells derived from human, mouse, or rat, with fibroblast, epithelial, neuronal, or myeloid characteristics. By contrast, GOLPH3L was expressed at detectable levels in only a subset of these lines, especially those with characteristics of secretory epithelia (HEK 293, LNCaP, MCF-7). However, quantification of GOLPH3 and GOLPH3L expression (Figure 3B) demonstrated that even among those lines that do express GOLPH3L, GOLPH3L is expressed at ∼3- to 10-fold lower levels than GOLPH3.</p>
              <figure data-origid="F3">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F3">FIGURE 3:</a>
                <figcaption>
                  <p id="s5s2f2p1">GOLPH3 and GOLPH3L expression patterns. (A) Pan-specific GOLPH3 (G3)/GOLPH3L (G3L) Western blot of SDS lysates obtained from various cell lines: HEK293 (human embryonic kidney), HeLa (human cervical carcinoma), LNCaP (human prostate carcinoma), MCF-7 (human breast adenocarcinoma), Neuro2A (mouse neuroblastoma), NIH 3T3 (mouse embryonic fibroblast), NRK (normal rat kidney epithelia), N-Tera 2 (pluripotent human embryonic carcinoma), RAW (mouse macrophage), and U2-OS (human osteosarcoma). GOLPH3 is expressed in all lines, whereas GOLPH3L is expressed in only a subset and at lower levels. (B) Quantification of relative expression levels of GOLPH3 and GOLPH3L in various cell lines. Integrated density was measured and background subtracted for GOLPH3, GOLPH3L, and GAPDH bands from Western blots performed in three independent experiments. Graphed are mean GOLPH3 and GOLPH3L levels normalized to GAPDH levels; error bars represent SEM. (C) Pan-specific GOLPH3/GOLPH3L Western blot of SDS lysates obtained from various tissues from a male A/J mouse (an inbred albino line, often used in cancer and immunology research). Multiple exposures of the blot are shown to allow observation of the range of GOLPH3 and GOLPH3L expression seen across tissues. ns, a nonspecific band observed on the longest exposure. GAPDH Western blot is provided to show relative protein loading. (D) Quantification of relative expression levels of GOLPH3 and GOLPH3L, normalized to GAPDH, in various mouse tissues. Results are representative of three replicates. GOLPH3 is ubiquitous, whereas GOLPH3L is expressed in only a subset of tissues. Salivary gland, small intestine, and skin (tail) express levels of GOLPH3L that approach or exceed levels of GOLPH3.</p>
                </figcaption>
              </figure>
              <p id="s5s2p4">We next examined the relative expression of GOLPH3 and GOLPH3L in SDS lysates of a variety of tissues harvested from a mouse of the inbred A/J strain. As shown in Figure 3C, we observed a range of generally high levels of GOLPH3 expression among all tissues examined. GOLPH3L is expressed in only a subset of tissues, with highest expression relative to GOLPH3 in salivary gland, small intestine, and skin (tail; Figure 3D), suggesting once more that expression of GOLPH3L is highest in secretory cells.</p>
              <p id="s5s2p5">The ubiquitous expression of GOLPH3 suggests that it serves a function common to all cells. Of note, in HeLa cells we observe expression of GOLPH3 but not of GOLPH3L. In these cells we previously demonstrated that GOLPH3 is required to maintain normal Golgi morphology and for anterograde trafficking from the Golgi to the plasma membrane (Dippold et al., 2009). Because GOLPH3L expression in most cell types is relatively low compared with that of GOLPH3, it seems likely that GOLPH3L must somehow function differently from GOLPH3.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s3h1">GOLPH3L binds PI4P and consequently localizes to the Golgi</h3>
            <div class="content">
              <p id="s5s3p1">Because the region of GOLPH3L that most highly resembles GOLPH3 is the domain that confers binding to PI4P, we examined whether GOLPH3L also binds selectively to any of the phosphoinositides. We expressed GOLPH3 or GOLPH3L by in vitro transcription and translation (Figure 4A) and assessed binding to each of the phosphoinositides using a lipid blot assay (Dowler et al., 2002; Dippold et al., 2009). We found that similar to GOLPH3, GOLPH3L binds preferentially to PI4P (Figure 4B).</p>
              <figure data-origid="F4">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F4">FIGURE 4:</a>
                <figcaption>
                  <p id="s5s3f1p1">GOLPH3L binds PI4P in vitro. (A) Autoradiogram of gel of 35S-labeled, in vitro–translated proteins, demonstrating production of proteins of appropriate molecular weight and similar levels of expression. (B) Lipid blot assay. In vitro–translated proteins were used to blot membranes spotted with each phosphoinositide at multiple concentrations. From high to low the amount of lipid spotted was 180, 60, and 20 pmol, respectively. Drosophila PX domain–containing protein CG5439 (as a control for specificity of the assay) binds preferentially to PI3P. Human GOLPH3 and GOLPH3L bind PI4P, whereas GOLPH3 PI4P-binding mutant R90L does not.</p>
                </figcaption>
              </figure>
              <p id="s5s3p2">PI4P is specifically enriched on the cytosolic face of the trans-Golgi (Nakagawa et al., 1996; Wong et al., 1997; Wang et al., 2003; Godi et al., 2004). It was previously established that GOLPH3 localizes to the trans-Golgi (Wu et al., 2000; Snyder et al., 2006) as a consequence of its interaction with PI4P (Dippold et al., 2009). We predicted that since GOLPH3L also binds preferentially to PI4P in vitro, it was likely to localize to the Golgi. To test this prediction, we expressed EGFP-GOLPH3L in HeLa cells and then compared its localization to known Golgi markers and to GOLPH3 (Figure 5). We found that EGFP-GOLPH3L colocalizes with the Golgi-resident proteins GM130, TGN46, and p230. When coexpressed, EGFP-GOLPH3L tightly colocalizes with mCherry-GOLPH3 at the Golgi. We observed similar colocalization when we used reciprocally tagged GOLPH3 and GOLPH3L.</p>
              <figure data-origid="F5">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F5">FIGURE 5:</a>
                <figcaption>
                  <p id="s5s3f2p1">GOLPH3L localizes to the Golgi. (Top three rows) EGFP-tagged human GOLPH3L (green) was expressed in HeLa cells, and then the cells were fixed and observed by IF with an antibody against cis-Golgi protein GM130 or trans-Golgi protein TGN46 or p230 (red) and costained with DAPI to mark the nucleus (blue). (Bottom two rows) EGFP- (green) or mCherry-tagged (red) GOLPH3 and GOLPH3L were coexpressed in HeLa cells. Images shown here and in all subsequent figures are maximum projections in Z of confocal Z-stacks. Bar, 10 μM. Inset, a magnified view of the corresponding boxed region.</p>
                </figcaption>
              </figure>
              <p id="s5s3p3">To determine whether localization of GOLPH3L to the Golgi requires GOLPH3, we expressed EGFP-GOLPH3L in cells transfected with either control siRNA or siRNA to specifically knock down GOLPH3 (Supplemental Figure S2). Even in cells in which endogenous GOLPH3 levels were reduced to &lt;10% of control cells, EGFP-GOLPH3L still localized to the Golgi. Therefore GOLPH3L can localize to the Golgi independently of GOLPH3.</p>
              <p id="s5s3p4">To determine whether GOLPH3L localization to the Golgi is dependent on PI4P, we examined GOLPH3L localization after depletion of PI4P from the Golgi. To deplete PI4P from the Golgi, we expressed the Sac1 PI4P phosphatase, specifically the K2A mutant fused to EGFP, which constitutively localizes to the Golgi (Rohde et al., 2003). It was shown previously that overexpression of Sac1-K2A depletes Golgi PI4P without perturbing other phosphoinositides (Blagoveshchenskaya et al., 2008; Dippold et al., 2009). When expressed together with EGFP-Sac1-K2A (but not EGFP alone) mCherry-GOLPH3L no longer localizes to the Golgi but is instead found diffusely throughout the cell (Figure 6). We conclude that similar to GOLPH3, GOLPH3L localizes to the Golgi as a consequence of its binding to PI4P.</p>
              <figure data-origid="F6">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F6">FIGURE 6:</a>
                <figcaption>
                  <p id="s5s3f3p1">Golgi localization of GOLPH3L is dependent on PI4P. HeLa cells were transiently transfected with expression vectors for mCherry-tagged GOLPH3L (red), together with EGFP-tagged PI4P phosphatase Sac1-K2A (a mutant that is constitutively localized to the Golgi; Rohde et al., 2003) or EGFP (control; green) and stained for the Golgi marker GM130 (blue). In control cells mCherry-GOLPH3L and GM130 colocalize at the Golgi, whereas in cells coexpressing EGFP-Sac1-K2A, GOLPH3L is found diffusely throughout the cell. Bar, 10 μm.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s4h1">Overexpression of GOLPH3L results in Golgi compaction</h3>
            <div class="content">
              <p id="s5s4p1">Knockdown or overexpression of GOLPH3 has dramatic effects on Golgi morphology (Dippold et al., 2009). To assess the function of GOLPH3L we examined the effect of perturbations of GOLPH3L expression on the Golgi. We chose to use HEK 293 cells for these experiments because they express relatively high levels of both GOLPH3 and GOLPH3L (Figure 3A) and are easily transfected. First, we overexpressed empty vector, GOLPH3, or GOLPH3L, used immunofluorescence (IF) staining of TGN46 to observe the Golgi, and measured the area of the Golgi (normalized to the area of the nucleus). We verified overexpression of GOLPH3 or GOLPH3L by Western blot (Figure 7A). We observed that control cells transfected with empty vector have a normal Golgi ribbon that extends partially around, and remains closely apposed to, the nucleus (Figure 7B). However, overexpression of GOLPH3 results in a more dispersed Golgi, consistent with the reported role for GOLPH3/MYO18A in exerting a tensile force upon the Golgi ribbon to assist in vesiculation (Dippold et al., 2009). Surprisingly, overexpression of GOLPH3L has an opposite effect on Golgi morphology, resulting in compaction of the Golgi. Quantification of the area of the Golgi demonstrates a significantly dispersed Golgi in response to overexpression of GOLPH3 but a significantly compacted Golgi in response to overexpression of GOLPH3L (Figure 7C). Therefore we conclude that when overexpressed in cells, GOLPH3 and GOLPH3L have opposite effects on Golgi morphology.</p>
              <figure data-origid="F7">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F7">FIGURE 7:</a>
                <figcaption>
                  <p id="s5s4f1p1">Overexpression of GOLPH3L causes Golgi compaction. HEK 293 cells were transiently transfected with expression vectors for GOLPH3, GOLPH3L, or empty vector and cotransfected with one-fifth as much expression vector for EGFP to mark transfected cells. (A) Demonstration by Western blot of overexpression of GOLPH3 or GOLPH3L together with EGFP in HEK 293 cells. Duplicate transfections are shown. Western blot to GAPDH provides an indication of relative total protein loading. (B) Representative IF images of cells overexpressing GOLPH3 or GOLPH3L compared with control. Transfected cells are detected by expression of cotransfected EGFP, shown in green. DAPI marks the nucleus, in blue. GOLPH3 IF is shown in red, and increased GOLPH3 staining is evident in GOLPH3-overexpressing cells. TGN46 staining, shown in magenta, marks the location of the Golgi. Compared to cells transfected with empty vector, GOLPH3-overexpressing cells have a more expanded Golgi, whereas GOLPH3L-overexpressing cells have a more compact Golgi. Bar, 10 μm. (C) Quantification of Golgi area of cells from B. The Golgi area for each cell was measured, normalized to the nuclear area, and expressed relative to control. GOLPH3 overexpression results in expansion of the Golgi compared with control cells. Conversely, GOLPH3L overexpression leads to a significant compaction of the Golgi compared with control cells. Graphed are mean and SEM. The number of cells measured (pooled from three independent experiments) and p values vs. the empty vector control (by unpaired t test) are indicated.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s5h1">Depletion of GOLPH3L results in Golgi dispersal</h3>
            <div class="content">
              <p id="s5s5p1">Next we examined the Golgi in cells depleted for GOLPH3 or GOLPH3L. Semiquantitative Western blotting demonstrates knockdown of GOLPH3L protein expression to &lt;25% of normal endogenous levels in HEK 293 cells using any of three independent siRNA oligos targeted against the GOLPH3L mRNA (Figure 8A). We compared the effects of knocking down GOLPH3 versus GOLPH3L on Golgi morphology (Figure 8B). As previously established, knockdown of GOLPH3 results in a compact Golgi (Dippold et al., 2009). Conversely, we found that knockdown of GOLPH3L using any of three independent siRNA oligos results in dispersal of the Golgi. This dispersal of the Golgi involves cis, medial, and trans compartments as shown by a cis marker (GM130), medial markers (α-mannosidase II and β-1,4-galactosyltransferase), and trans markers (GOLPH3, TGN46, and α-2,6-sialyltransferase; Figure 8B; also see Figure 10B and Supplemental Figures S4 and S5). Despite the dispersal of the Golgi, in all cases the cis, medial, and trans markers remain colocalized. Quantification of the area of the Golgi confirmed that these changes in the Golgi were highly significant (Figure 8C). Thus we conclude that GOLPH3 and GOLPH3L have opposing roles in maintaining Golgi morphology.</p>
              <figure data-origid="F8">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F8">FIGURE 8:</a>
                <figcaption>
                  <p id="s5s5f1p1">GOLPH3L functions to prevent Golgi dispersal. (A) Three independent siRNAs knock down GOLPH3L. Semiquantitative Western blots of duplicate samples of cells transfected with control siRNA, siRNA specific to GOLPH3, or three independent siRNAs targeted to GOLPH3L. Lysates from control cells are loaded at different relative amounts to provide a standard curve to allow semiquantitative assessment of knockdown efficiency. Pan-specific GOLPH3 Western blot shows &gt;90% knockdown of GOLPH3 and &gt;75% knockdown of GOLPH3L by each of the three siRNAs compared with cells treated with control siRNA. Western blot to GAPDH demonstrates approximately equal loading (for samples with 100% relative loading). (B) GOLPH3L knockdown causes expansion of the Golgi. Shown are representative IF images of cells transfected with control siRNA, siRNA to GOLPH3, or three independent siRNAs to GOLPH3L. GOLPH3 IF (shown in green) is reduced upon GOLPH3 knockdown but not GOLPH3L knockdown. GM130 staining (shown in red) to mark the Golgi shows compaction upon knockdown of GOLPH3 and expansion upon knockdown of GOLPH3L with each of the three siRNAs. Bar, 10 μm. (C) Quantification of Golgi area in siRNA-transfected cells, measured as indicated for Figure 7. Knockdown of GOLPH3 results in compaction of the Golgi, whereas knockdown of GOLPH3L by each of the three siRNAs results in expansion of the Golgi. Differences are highly statistically significant, with number of cells measured (pooled from three independent experiments) and p values (by unpaired t test) as indicated.</p>
                </figcaption>
              </figure>
              <figure data-origid="F9">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F9">FIGURE 9:</a>
                <figcaption>
                  <p id="s5s5f2p1">GOLPH3L does not affect Golgi morphology in cells deficient of GOLPH3 or MYO18A. (A) Western blots showing knockdown of GOLPH3, GOLPH3L (G3L), and MYO18A each alone or in combination. Lysates from control siRNA–treated cells are loaded at different relative amounts to provide a standard curve for semiquantitative assessment of knockdown efficiency. Tuberin and GAPDH Western blots demonstrate approximately equal loading. (B) Knockdown of GOLPH3L causes Golgi dispersal. Knockdown of GOLPH3 or MYO18A causes Golgi compaction. Knockdown of GOLPH3L has no effect in cells knocked down for GOLPH3 or MYO18A. Supplemental Figure S3 provides representative images for cells transfected with GOLPH3L siRNA oligos 2 or 3. Bar, 10 μm. (C) Quantification of Golgi area in knockdown cells, measured as indicated for Figure 7. Data are pooled from three independent experiments. The Golgi is significantly expanded compared with controls when GOLPH3L is knocked down, whereas when GOLPH3 or MYO18A is knocked down, the Golgi is significantly more compact than controls. When GOLPH3L knockdown is combined with GOLPH3 or MYO18A knockdown, the Golgi is again compact, similar to GOLPH3 or MYO18A knockdown alone.</p>
                </figcaption>
              </figure>
              <figure data-origid="F10">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F10">FIGURE 10:</a>
                <figcaption>
                  <p id="s5s5f3p1">GOLPH3 family members are dispensable for Golgi localization of Sial-EGFP. (A) Western blot demonstrates knockdown of GOLPH3L and GOLPH3 in HEK 293 cells expressing Sial-EGFP. Lysates from control siRNA–treated cells are loaded at different relative amounts to provide a standard curve for semiquantitative assessment of knockdown efficiency. GAPDH Western blot demonstrates approximately equal loading. (B) HEK 293 cells expressing Sial-EGFP (green) were fixed and stained for IF to GOLPH3 (magenta), GM130 to mark the Golgi (red), and DAPI to label the nucleus (blue). The Golgi is expanded compared with controls when GOLPH3L is knocked down, and the Golgi is condensed compared with controls when GOLPH3 is knocked down. However, in all cases Sial-EGFP remains tightly localized to the Golgi. Bar, 10 μm. Inset, a magnified view of the corresponding boxed region. Similar results with α-mannosidase II-EGFP and β-1,4-galactosyltransferase-EYFP are included in Supplemental Figures S4 and S5.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s6h1">Epistasis relationships between GOLPH3L and GOLPH3/MYO18A</h3>
            <div class="content">
              <p id="s5s6p1">The opposite effects of GOLPH3 and GOLPH3L on Golgi morphology could be explained either by GOLPH3 opposing an inward-directed centripetal influence of GOLPH3L or by GOLPH3L opposing an outward-directed, centrifugal, tensile influence by GOLPH3. To determine whether a centripetal or a centrifugal force on the Golgi is primary, we sought to order GOLPH3 and GOLPH3L in a genetic pathway using epistasis analysis of the siRNA knockdowns. To assess epistasis relationships between GOLPH3L and the GOLPH3/MYO18A pathway, we combined GOLPH3L knockdown with either GOLPH3 or MYO18A knockdown. Western blotting demonstrates efficient knockdown of GOLPH3 or MYO18A each alone or together with GOLPH3L (Figure 9A). We assessed Golgi morphology by IF using GM130 as a Golgi marker (Figure 9B and Supplemental Figure S3). As expected, knockdown of GOLPH3 or MYO18A each resulted in compaction of the Golgi. Knockdown of GOLPH3L alone with any of three siRNA oligos resulted in dispersal of the Golgi. However, knockdown of GOLPH3L had no effect on the compact Golgi in cells in which GOLPH3 or MYO18A are also knocked down. Quantification of Golgi area demonstrates clearly that GOLPH3L becomes irrelevant in the absence of GOLPH3 or MYO18A (Figure 9C). Similarly, we note that depletion of PI4P by Sac1 (Figure 6), which causes both GOLPH3 and GOLPH3L to dissociate from the Golgi, also produces a compact Golgi. From these results we conclude that the effects of GOLPH3L on Golgi morphology require GOLPH3 and MYO18A and that the outward-directed tensile force exerted by GOLPH3 and MYO18A is primary.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s7h1">Role of GOLPH3/GOLPH3L in glycosyltransferase localization</h3>
            <div class="content">
              <p id="s5s7p1">The yeast homologue of GOLPH3, Vps74p, is required for proper Golgi localization of several mannosyltransferase proteins in budding yeast (Tu et al., 2008; Schmitz et al., 2008). These mannosyltransferases are peculiar to yeast, with no homologues among higher organisms, and recent evidence suggests that the role of Vps74p in their localization might be quite indirect (Wood et al., 2012). In the myeloid cell line KG1a, knockdown of GOLPH3 is reported to alter the Golgi localization of core 2 N-acetylglucosaminyltransferase 1 (Ali et al., 2012). However, in HeLa cells proper localization to the Golgi of several other glycosyltransferases is not affected by knockdown of GOLPH3 (Dippold et al., 2009). We wondered whether GOLPH3L alone or in concert with GOLPH3 might influence glycosyltransferase localization in cells that express both GOLPH3 paralogues, namely HEK 293 cells. We found that knockdown of GOLPH3, GOLPH3L, or both GOLPH3 and GOLPH3L in combination had no effect on the Golgi localization of several fluorescently tagged glycosyltransferases. In particular, α-2,6-sialyltransferase-EGFP (Sial-EGFP), α-mannosidase II-EGFP (ManII-EGFP), and β-1,4-galactosyltransferase-EYFP (GalT-EYFP) localized equally to the Golgi regardless of knockdown of GOLPH3, GOLPH3L, or both (Figure 10 and Supplemental Figures S4 and S5). In mammalian cells, localization of glycosyltransferases to the Golgi is not generally dependent on GOLPH3 or GOLPH3L.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s8h1">GOLPH3 and GOLPH3L differ in their ability to interact with MYO18A</h3>
            <div class="content">
              <p id="s5s8p1">GOLPH3 function at the Golgi is dependent on its interaction with MYO18A (Dippold et al., 2009). Because GOLPH3L binds to PI4P and localizes to the Golgi similarly to GOLPH3, but the two proteins clearly differ in their function at the Golgi, we investigated whether they differ in their ability to interact with MYO18A. To test the interaction with MYO18A we performed specific immunoprecipitations (IPs) of endogenous GOLPH3 or GOLPH3L from HEK 293 cell lysates and assayed for coIP of endogenous MYO18A. We prepared HEK 293 cell lysates and immunoprecipitated with preimmune serum, our anti-GOLPH3 antiserum, or a commercial antibody to GOLPH3L (Figure 11A). We then performed Western blots on the input, IP, and post-IP supernatants with our anti-GOLPH3 antiserum. Although we know that when used for Western blotting our anti-GOLPH3 antiserum detects GOLPH3 and GOLPH3L equally well (Figure 2A), we observe that when used for IP it is specific for GOLPH3. In particular, with our anti-GOLPH3 antiserum we observe robust IP of GOLPH3, with nearly quantitative depletion from the post-IP supernatant, but no detectable IP of GOLPH3L and no depletion of GOLPH3L from the post-IP supernatant (Figure 11A). By contrast, the antibody to GOLPH3L robustly immunoprecipitates GOLPH3L, with quantitative depletion from the post-IP supernatant, but no IP of GOLPH3 and no depletion of GOLPH3 from the post-IP supernatant. We conclude that we can use these reagents to specifically immunoprecipitate GOLPH3 or GOLPH3L.</p>
              <figure data-origid="F11">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F11">FIGURE 11:</a>
                <figcaption>
                  <p id="s5s8f1p1">GOLPH3L does not share GOLPH3’s tight interaction with MYO18A. (A) IP of endogenous proteins from HEK 293 cell lysates using antibodies for GOLPH3, GOLPH3L, or preimmune serum. Preimmune serum fails to immunoprecipitate GOLPH3, GOLPH3L, or MYO18A. GOLPH3 (but not GOLPH3L) is specifically immunoprecipitated using GOLPH3 antibody, and MYO18A coimmunoprecipitates with GOLPH3. GOLPH3L (but not GOLPH3) is specifically immunoprecipitated using GOLPH3L antibody; however, MYO18A does not appreciably coimmunoprecipitate with GOLPH3L. (B) IP using antibodies to GOLPH3 or GOLPH3L from control or GOLPH3 knockdown lysates from HEK 293 cells. In control lysates GOLPH3 (but not GOLPH3L) is specifically immunoprecipitated using GOLPH3 antibody, and MYO18A coimmunoprecipitates with GOLPH3. GOLPH3L (but not GOLPH3) is specifically immunoprecipitated using GOLPH3L antibody; however MYO18A does not coimmunoprecipitate with GOLPH3L. In GOLPH3-knockdown lysates GOLPH3 levels are greatly reduced. Despite reduced GOLPH3 levels, specific IP of GOLPH3L still does not appreciably coimmunoprecipitate MYO18A.</p>
                </figcaption>
              </figure>
              <p id="s5s8p2">We also probed the Western blot with an antibody that we have documented to be specific for MYO18A (Dippold et al., 2009; also Figure 9). As expected, in the input we observe two bands at ∼230 and ∼190 kDa corresponding to known MYO18A splice variants (Mori et al., 2003). IP of GOLPH3 robustly coimmunoprecipitates MYO18A, with significant depletion from the post-IP supernatant, as expected (Dippold et al., 2009). By contrast, although the IP of GOLPH3L robustly immunoprecipitates GOLPH3L, there is little or no detectable coIP of MYO18A and no depletion of MYO18A from the post-IP supernatant.</p>
              <p id="s5s8p3">Because GOLPH3 is more abundant than GOLPH3L, we wondered whether GOLPH3 might saturate available MYO18A, outcompeting GOLPH3L. To determine whether GOLPH3L might be able to interact with MYO18A if this competition were reduced, we used siRNA to knock down GOLPH3. HEK 293 cells were transfected with control siRNA or siRNA specific to GOLPH3. As shown in the input lane (Figure 11B), knockdown of GOLPH3 significantly reduced GOLPH3 protein levels. Of importance, in the knockdown the amount of GOLPH3L protein now exceeds the amount of GOLPH3 protein. As expected, IP of GOLPH3 from control lysates again robustly coimmunoprecipitates MYO18A. In addition, as shown previously, IP of GOLPH3L coimmunoprecipitated little or no MYO18A. Furthermore, IP of GOLPH3L from GOLPH3 knockdown lysates also did not significantly coimmunoprecipitate MYO18A. Very long exposure of the blots revealed a faint band with the GOLPH3L immunoprecipitate, and therefore we cannot exclude the presence of a very weak interaction. Nevertheless, we conclude that in contrast to GOLPH3, GOLPH3L is largely unable to interact with MYO18A.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s5s9h1">GOLPH3 and GOLPH3L are both required for efficient secretion</h3>
            <div class="content">
              <p id="s5s9p1">GOLPH3 is required for efficient trafficking from the Golgi to the plasma membrane (Dippold et al., 2009). To further analyze the role of the GOLPH3 pathway in secretion, we used pulse-chase analysis to examine total secretory flux in cells knocked down for GOLPH3 pathway components. To measure total secretory flux, we pulse labeled HEK 293 cells with [35S]methionine and then chased them to examine total protein secretion into the media. As expected, treatment with the Golgi poison brefeldin A dramatically impaired overall protein secretion (Figure 12). Similarly, siRNA knockdown of GOLPH3 or MYO18A each reduced total secretory flux to nearly the same extent as treatment with brefeldin A. Finally, knockdown of GOLPH3L similarly impaired overall secretion. We conclude that although GOLPH3L is apparently dispensable in cell types that do not express it, in HEK 293 cells GOLPH3L, GOLPH3, and MYO18A are all required for efficient secretion.</p>
              <figure data-origid="F12">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F12">FIGURE 12:</a>
                <figcaption>
                  <p id="s5s9f1p1">Efficient secretion in HEK 293 cells is dependent on both GOLPH3 and GOLPH3L. In HEK 293 cells, a pulse-chase assay was used to measure total protein secretion as described in Materials and Methods. Graph shows mean secretion relative to control siRNA–treated cells with SEM and number of replicates (pooled from three independent experiments) as indicated. Control cells treated with brefeldin A, as well as cells with siRNA knockdown of GOLPH3, MYO18A, or GOLPH3L, all have significantly decreased total secretory flux compared with control. Statistical significance is indicated (t test).</p>
                </figcaption>
              </figure>
            </div>
          </section>
        </div>
      </section>
      <section data-type="discussion">
        <h2 id="s6h1">DISCUSSION</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s6s1h1">GOLPH3 and GOLPH3L share some basic functional similarities</h3>
            <div class="content">
              <p id="s6s1p1">Previously, little was known about GOLPH3L, other than its high degree of homology to GOLPH3. As might be predicted by the sequence homology, we find that the two proteins do have some functional similarities. The domain in GOLPH3 that is responsible for binding to PI4P is highly conserved in GOLPH3L. In particular, the key residues that interact with PI4P, namely R90, W81, R171, and R174 (Dippold et al., 2009; Wood et al., 2009), are conserved between the two proteins (Supplemental Figure S1). As might be predicted from this sequence conservation, we find that GOLPH3L also binds tightly and specifically to PI4P.</p>
              <p id="s6s1p2">Proteins that demonstrate tight, specific binding to PI4P in vitro are found to localize to the Golgi in cells (Levine and Munro, 2002; Hanada et al., 2003; Godi et al., 2004). For human GOLPH3 and the S. cerevisiae homologue Vps74p we demonstrated that localization to the Golgi depends on the interaction of the protein with PI4P at the Golgi (Dippold et al., 2009). Again, as might be expected from the interaction of GOLPH3L with PI4P, we observe that GOLPH3L also localizes to the Golgi in cells, and this Golgi localization depends on PI4P.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s2h1">GOLPH3 and GOLPH3L differ significantly</h3>
            <div class="content">
              <p id="s6s2p1">Despite their similarities, GOLPH3L does not function as a duplicate version of GOLPH3. GOLPH3L crucially lacks the ability to interact tightly with MYO18A that is critical for the function of GOLPH3. Thus GOLPH3L can interact with GOLPH3’s upstream regulator, PI4P, but lacks the ability to link to GOLPH3’s effector, MYO18A, features that could be expected for a protein that functions as a dominant negative. Indeed, GOLPH3L acts in opposition to GOLPH3 in determining Golgi morphology, and its ability to act on the Golgi is dependent on GOLPH3 and MYO18A, as illustrated in the model in Figure 13.</p>
              <figure data-origid="F13">
                <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3596250/#F13">FIGURE 13:</a>
                <figcaption>
                  <p id="s6s2f1p1">Model depicting the relationship between PI4P, GOLPH3, MYO18A, and GOLPH3L at the Golgi. GOLPH3 binds PI4P at the trans face of the Golgi. GOLPH3 interacts with MYO18A, which then provides a link to the actin cytoskeleton and a mechanism to provide tension to assist in vesicle budding, also serving to stretch the Golgi into its characteristic shape. GOLPH3L also binds PI4P but not MYO18A. GOLPH3L functions to antagonize GOLPH3 and MYO18A.</p>
                </figcaption>
              </figure>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s3h1">How does GOLPH3L oppose GOLPH3?</h3>
            <div class="content">
              <p id="s6s3p1">The opposite effects observed on Golgi morphology from perturbing GOLPH3 versus GOLPH3L expression, together with the results of our epistasis analysis, argue strongly that GOLPH3L functions as an inhibitor of GOLPH3. Nevertheless, there remains an important question as to how a small amount of GOLPH3L can significantly interfere with a much larger amount of GOLPH3. For example, in HEK 293 cells the concentration of GOLPH3L is only ∼15% of that of GOLPH3. In other cells the disparity is even larger (Figure 3B). A straightforward model for the ability of GOLPH3L to act as a dominant negative would be for it to compete with GOLPH3 for binding to PI4P, essentially displacing GOLPH3 from the Golgi and thus interfering with the linkage of the Golgi to MYO18A. For this model to be true, it would require that GOLPH3 and GOLPH3L be found in excess of PI4P at the Golgi. Although GOLPH3 is an abundant protein, PI4P is likely to be significantly more abundant. Moreover, as shown in Figure 5, simultaneous overexpression of both GOLPH3 and GOLPH3L is insufficient to saturate binding and localization to the Golgi. Nevertheless, it remains possible that in the context of other PI4P-binding proteins, a small reduction in the amount of available PI4P due to binding to GOLPH3L is sufficient to shift the localization of GOLPH3 enough to perturb the function of a finely tuned system.</p>
              <p id="s6s3p2">An alternative to explain how a small amount of GOLPH3L can significantly interfere with the function of a much larger amount of GOLPH3 would be to suggest that the two proteins may heterodimerize and that incorporation of GOLPH3L into a heterodimer renders the complex nonfunctional. Indeed, the crystal structures of yeast Vps74p and human GOLPH3 both indicate homodimerization (Schmitz et al., 2008; Wood et al., 2009), albeit under conditions that are highly nonphysiological. Unfortunately, we have been unable to detect in cells either homodimerization of GOLPH3 or heterodimerization of GOLPH3 and GOLPH3L. Experimental detection of their interaction is complicated by the observation that tagging GOLPH3 at either end, even with a small epitope tag, renders the protein nonfunctional (unpublished data; also compare Figure 5, in which overexpression of tagged proteins has no effect on normal Golgi morphology, to Figure 7, in which overexpression of native proteins can elicit changes such as Golgi compaction or Golgi dispersal). Of importance, Figure 11 demonstrates that IPs of GOLPH3 that robustly coimmunoprecipitate MYO18A fail to coimmunoprecipitate any detectable GOLPH3L and vice versa. Nevertheless, it remains possible that when bound to PI4P and constrained to the two-dimensional Golgi membrane a weak interaction is sufficient to enable dimerization of the proteins. Because of the experimental constraints, better testing of this model will require more complex experimental designs.</p>
              <p id="s6s3p3">A third alternative explanation for the ability of a limited amount of GOLPH3L to oppose the function of GOLPH3/MYO18A would be to posit that GOLPH3L might have its own set of effectors that have yet to be identified and that these allow amplification of the response to a small amount of GOLPH3L. Our evidence neither supports nor refutes this possibility. Clearly, additional data would be needed to lend credence to this model.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s4h1">How might GOLPH3L function in secretory tissues?</h3>
            <div class="content">
              <p id="s6s4p1">We find that GOLPH3L is expressed at appreciable levels only in a subset of cell lines and tissue types that we investigated. Of note, the tissues that most highly express GOLPH3L are those that are highly secretory, such as salivary gland, small intestine, and skin. Clearly, in cells that do not express GOLPH3L that protein is not required for secretion. However, our data indicate that in cells that exhibit high levels of secretion, such as HEK 293 cells, GOLPH3L is expressed and is required for secretory function. There are several possible explanations for these results. One possibility is that GOLPH3L might have additional effectors apart from GOLPH3/MYO18A that serve a positive role required for secretion but only in certain cell types due to the presence (or absence) of other unknown factors. We do not have evidence for the existence of these hypothetical effectors. Additional evidence would be needed to consider this model further.</p>
              <p id="s6s4p2">Alternatively, in cells with high secretory activity, the unopposed GOLPH3/MYO18A complex drives secretion, leading to the production of a dispersed Golgi, a form that we hypothesize might interfere with continued optimal trafficking. Thus in highly secretory cells GOLPH3L would serve as a throttle to prevent overexuberant secretion and thus, by opposing GOLPH3/MYO18A, maintains a normal Golgi morphology that allows continuously high (but not too high) rates of secretion. According to this model, loss of GOLPH3L would result initially in a burst of elevated secretion, with the consequence of dispersal of the Golgi in response to unopposed GOLPH3/MYO18A. The dispersal of the Golgi would then result in subsequently impaired secretion. This model is attractive in that it is consistent with the known roles at the Golgi of GOLPH3, MYO18A, and GOLPH3L. Testing of this model will depend on the development of conditional alleles that allow rapid activation or inactivation of GOLPH3 or GOLPH3L.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s6s5h1">GOLPH3 is the primordial version</h3>
            <div class="content">
              <p id="s6s5p1">Because GOLPH3 and GOLPH3L function oppositely in regulating Golgi morphology, we can wonder which best represents the primordial function of the single GOLPH3 found in lower organisms. Before the present study, the high degree of homology among all GOLPH3 homologues, with a similar distance from yeast Vps74p to human GOLPH3 or GOLPH3L, provided no insight into which is the primordial version of GOLPH3. However, we note that among vertebrates the GOLPH3 proteins have diverged from each other significantly less than GOLPH3L proteins have diverged from each other. For example, the amino acid sequence distance (Felsenstein, 1996) from human to mouse for GOLPH3 is 0.031, whereas for GOLPH3L it is 0.112 (Figure 1B). This argues that GOLPH3 is under higher evolutionary sequence constraint, suggesting that GOLPH3 has more functional interactions than GOLPH3L. This difference in evolutionary sequence constraint is consistent with the observation that GOLPH3 interacts tightly with MYO18A, whereas GOLPH3L does not.</p>
              <p id="s6s5p2">In our study we find that GOLPH3 is ubiquitously expressed in all cell types, whereas only a few tissues express GOLPH3L and always together with GOLPH3. Furthermore, we find that the ability of GOLPH3L to determine Golgi morphology requires an intact GOLPH3/MYO18A pathway, allowing it to serve as an antagonist. Taken together, these results argue that GOLPH3 is the primordial version and that the functional interaction that distinguishes them—interaction with a myosin motor—is likely common to the function of GOLPH3 homologues found in lower organisms. Our data indicate that the evolutionary invention of GOLPH3L in higher organisms provides an additional point of control to modulate Golgi structure and function in highly secretory cells.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="materials|methods">
        <h2 id="s7h1">MATERIALS AND METHODS</h2>
        <div class="subsections">
          <section data-type="">
            <h3 id="s7s1h1">Cell culture</h3>
            <div class="content">
              <p id="s7s1p1">Mammalian cell lines, including HEK 293 (AD-293 variant; Stratagene, Santa Clara, CA), NIH 3T3 fibroblasts, HeLa S3, LNCaP, MCF-7, Neuro2A, NRK, N-Tera 2, RAW macrophages, and U2-OS cells, were grown according to American Type Culture Collection (Manassas, VA) guidelines. For transient transfection of DNA into mammalian cells we used linear polyethyleneimine MW25K (Boussif et al., 1995).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s2h1">Antibodies</h3>
            <div class="content">
              <p id="s7s2p1">GOLPH3 antibody was previously described (Dippold et al., 2009). GOLPH3L antibody (26-40) was purchased from Sigma-Aldrich (St. Louis, MO). GAPDH antibody and GFP antibody (B-2) were from Santa Cruz Biotechnology (Santa Cruz, CA). Tuberin and GAPDH antibodies were from Cell Signaling Technology (Beverly, MA). TGN46 antibody was purchased from Serotec (Raleigh, NC). GM130 and p230 antibodies were from BD Biosciences (San Diego, CA). MYO18A antibody was a generous gift from Zissis Chroneos (University of Texas, Tyler, TX). Horseradish peroxidase (HRP) conjugated goat anti-rabbit immunoglobulin secondary antibody for Western blot was from Rockland (Gilbertsville, PA). HRP-conjugated horse anti-mouse immunoglobulin secondary antibody for Western blot was from Cell Signaling Technology. HRP-conjugated mouse anti-rabbit immunoglobulin secondary antibody for Western blot was from Jackson ImmunoResearch (West Grove, PA). Alexa Fluor (488, 594, and 647)–conjugated secondary antibodies for IF were from Invitrogen (Carlsbad, CA).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s3h1">Cloning/vectors</h3>
            <div class="content">
              <p id="s7s3p1">Human GOLPH3L was amplified from the Mammalian Gene Collection clone IMAGE ID 3837466. Sequences of primers used to clone GOLPH3L into pcDNA3.1(-) (Invitrogen), pEGFP-C1 (BD Biosciences), and mCherry-C1 (a gift from Roger Tsien) were as follows: forward (adding EcoRI site), 5′-GGTCAGGAATTCTGCCATGACCACTTTAACTCACCGGG-3′, and reverse (adding BamHI site), 5′-GTGTGGATCCCTAAGATTTATTGAAGGCTGCCAGCAC-3′. To make pcDNA3.1(-)-GOLPH3L, pEGFP–C1-GOLPH3L, and mCherry–C1-GOLPH3L, insert and vectors that had been double digested with EcoRI and BamHI were ligated together. The correct insert size and sequence were verified by restriction digest and sequencing. GOLPH3 and GOLPH3 R90L vectors have been described previously (Dippold et al., 2009).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s4h1">Lipid blots</h3>
            <div class="content">
              <p id="s7s4p1">Lipids (Cell Signals, Columbus, OH; and Echelon Biosciences, Salt Lake City, UT) were dissolved in dimethyl sulfoxide/20% CHCl3/50 mM HCl and validated by thin-layer chromatography, and 100 nl was spotted onto polyvinylidene fluoride (PVDF). Blots blocked overnight in 3% fatty acid–free bovine serum albumin (BSA) in Tris-buffered saline and Tween-20 (TBST; 150 mM NaCl, 50 mM Tris, pH 7.5, 0.03% Tween-20) were probed in refreshed block with 35S-labeled protein from a 25-μl in vitro transcription/translation mix (TNT Quick Coupled Transcription/Translation System; Promega, Madison, WI) with 33 μCi of translabel, reserving 2.5 μl for SDS–PAGE to verify expression. After 2 h, blots were washed five times in TBST and exposed to phosphorimager (Molecular Dynamics, Sunnyvale, CA).</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s5h1">Western blots</h3>
            <div class="content">
              <p id="s7s5p1">Cells and tissues were harvested, lysed in sample buffer (70 mM Tris-HCl, pH 6.8, 14% glycerol, 14% β-mercaptoethanol, 3.3% SDS, 0.04% bromophenol blue), boiled, and then resolved by SDS–PAGE. Proteins were transferred to PVDF and then blocked in TBST (14 mM Tris, pH 7.5, 150 mM NaCl, 0.05% Tween-20) with 5% nonfat dry milk for 10 min. Before primary antibody incubation, blots were washed three times for 5 min each in TBST. Primary antibodies were diluted in TBST with 5% nonfat dry milk as follows: rabbit anti-GOLPH3 1:5000, rabbit anti-GAPDH 1:5000, mouse anti-GFP 1:1000, and rabbit anti-MYO18A 1:1000. Blots were incubated in primary antibody overnight. Before secondary antibody incubation, blots were washed three times for 5 min each in TBST. Secondary antibodies were diluted in TBST with 5% nonfat dry milk as follows: HRP-goat anti-rabbit immunoglobulin 1:5000, HRP-horse anti-mouse immunoglobulin 1:5000, and HRP-mouse anti-rabbit immunoglobulin 1:10,000. Blots were incubated in secondary antibody for 2 h and then washed three times for 5 min each in TBST. Blots were soaked for 30 s in equal parts of enhanced chemiluminescence reagents 1 (100 mM Tris, pH 8.8, 2.5 mM luminal, 396 µM p-coumaric acid) and 2 (100 mM Tris, pH 8.8, 0.018% hydrogen peroxide) and then wrapped in clear plastic and used to expose films. Primary antibody step was performed at 4°C, but all other steps were carried out at room temperature.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s6h1">Immunofluorescence staining</h3>
            <div class="content">
              <p id="s7s6p1">Cells grown on glass coverslips were fixed for 20 min in phosphate-buffered saline (PBS; 137 mM NaCl, 2.68 mM KCl, 4.29 mM Na2HPO4, 1.76 mM KH2PO4, pH 7.4) containing 3.7% paraformaldehyde and then given three 5-min washes in PBS. Cells were permeabilized in PBS with 0.2% Triton X-100 for 10 min, followed by three 5-min washes in PBS. Cells were then blocked for 20 min in PBS with 8% BSA and 0.2% Tween-20, followed by three 5-min washes in PBS. Cells were incubated overnight in primary antibodies diluted in blocking solution at the following dilutions: rabbit anti-GOLPH3, 1:500; mouse anti-GM130, 1:1000; mouse anti-p230, 1:1000; and sheep anti-TGN46, 1:200. Before secondary antibody incubation, cells were washed three times for 5 min each in PBS. Cells were incubated for 2 h in secondary antibodies diluted in blocking solution with 0.1 μg/ml 4′,6-diamidino-2-phenylindole (DAPI) at the following dilutions: Alexa 594 anti-sheep, 1:500; Alexa 594 anti-mouse, 1:500; Alexa 647 anti-rabbit, 1:1000; and Alexa 647 anti-mouse, 1:1000. After secondary antibody incubation, cells were washed three times for 5 min each in PBS, and then the coverslips were mounted onto slides with Fluoromount-G (Southern Biotech, Birmingham, AL) and sealed with nail polish. Primary antibody step was performed at 4°C, but all other steps were carried out at room temperature.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s7h1">Fluorescence microscopy</h3>
            <div class="content">
              <p id="s7s7p1">Fluorescence microscopy was performed on an IX81-ZDC spinning-disk confocal microscope (Olympus, Tokyo, Japan) and analyzed with Slidebook (Intelligent Imaging Innovations, Denver, CO) and ImageJ (National Institutes of Health, Bethesda, MD) software. All images shown are maximum projections in Z of confocal Z-stacks.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s8h1">siRNA knockdown</h3>
            <div class="content">
              <p id="s7s8p1">siRNA oligos were synthesized by Invitrogen and transfected with RNAiMAX according to the manufacturer's instructions.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s9h1">siRNA oligonucleotide sequences</h3>
            <div class="content">
              <p id="s7s9p1">siRNA oligonucleotides containing Stealth modifications were purchased from Invitrogen. Oligonucleotides used to knock down GOLPH3 and MYO18A were GOLPH3-3 and MYO18A-1, as previously described (Dippold et al., 2009). Negative control oligonucleotide is a random, 48% GC sequence (catalog number 12935300; Invitrogen). The sequences of specific oligonucleotides were as follows: GOLPH3L-1 5′-GGUGGGUAAAUGACCCUCAGCGUAU-3′GOLPH3L-2 5′-CGUCGCACUGAAAUAAGCAAGAACU-3′GOLPH3L-3 5′-GAUGGAAAGUGAGGAAGACAGUAAU-3′</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s10h1">Secretion assay</h3>
            <div class="content">
              <p id="s7s10p1">HEK293 cells were labeled with 35S (EXPRESS35S Protein Labeling Mix; PerkinElmer) in DMEM without l-methionine, l-cystine, and l-glutamine and supplemented with 10% fetal bovine serum (FBS) for 30 min at 37°C. The cells were washed and chased at 37°C in unlabeled DMEM with 10% FBS. After a 10-min chase, conditioned media was collected, and cells were lysed in RIPA buffer. Proteins were precipitated from conditioned media and cell lysate samples with 10% trichloroacetic acid, and samples were quantified by scintillation counting. Brefeldin A–treated control cells were incubated with 10 μg/ml brefeldin A for 30 min before chase.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s11h1">Immunoprecipitations</h3>
            <div class="content">
              <p id="s7s11p1">HEK 293 cells lysed on ice in 150 mM NaCl, 10 mM NaHPO4, 2 mM EDTA, 10 mM 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate, and 5 mM dithiothreitol plus protease inhibitors were cleared and incubated with rabbit anti-GOLPH3, rabbit preimmune serum, or rabbit anti-GOLPH3L, precipitated with protein A–Sepharose (GE Healthcare, Piscataway, NJ), washed extensively, and boiled in SDS sample buffer.</p>
            </div>
          </section>
          <section data-type="">
            <h3 id="s7s12h1">Sequence alignments</h3>
            <div class="content">
              <p id="s7s12p1">EMBL-EBI ClustalW2 tool (Larkin et al., 2007; Goujon et al., 2010) was used to align Vps74p, GOLPH3, and GOLPH3L protein sequences from multiple species, as indicated in the legend to Supplemental Figure S1. Default settings for protein weight matrix, gaps, iterations, numiter, and clustering were selected.</p>
            </div>
          </section>
        </div>
      </section>
      <section data-type="supplementary-material">
        <h2 id="s8h1">Supplementary Material</h2>
      </section>
    </article>
  </body>
</html>
